<!doctype html>
<html class="no-js" lang="en">

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-134228602-13"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-134228602-13');
  </script>

  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>üöπ üßôüèø üßïüèæ What can be done in 48 hours? Interview with the BioHack 2019 bioinformatics hackathon winner üöå ‚úäüèº üòå</title>
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <meta name="description" content="The fourth bioinformatics hackathon BioHack 2020 starts on March 27 in St. Petersburg . During the existence of the hackathon, more than 300 young spe...">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <link rel="stylesheet" href="../../css/main.css">

  <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">

  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script>window.jQuery || document.write('<script src="../../js/vendors/jquery-3.3.1.min.js"><\/script>')</script>

  <script>document.write('<script src="//pagea' + 'd2.googles' + 'yndication.com/pagea' + 'd/js/a' + 'dsby' + 'google.js"><\/script>')</script>
  <script>
        var superSpecialObject = {};
        superSpecialObject['google_a' + 'd_client'] = 'ca-p' + 'ub-6974184241884155';
        superSpecialObject['enable_page_level_a' + 'ds'] = true;
       (window['a' + 'dsbygoogle'] = window['a' + 'dsbygoogle'] || []).push(superSpecialObject);
  </script>
</head>

<body>
  <!--[if lte IE 9]>
    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
  <![endif]-->
  <header class="page-header js-page-header">
    <a class="page-header-logo-container" href="https://geek-week.github.io/index.html"></a>
    <div class="page-header-text">Get best of the week</div>
  </header>
  <section class="page js-page"><h1>What can be done in 48 hours? Interview with the BioHack 2019 bioinformatics hackathon winner</h1><div class="post__body post__body_full">
      <div class="post__text post__text-html post__text_v1" id="post-content-body" data-io-article-url="https://habr.com/ru/company/epam_systems/blog/488034/"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">The fourth bioinformatics hackathon </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">BioHack 2020</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> starts on March 27 in St. Petersburg </font><font style="vertical-align: inherit;">. During the existence of the hackathon, more than 300 young specialists from different countries participated in it and 58 solutions were developed. Leading research organizations presented their projects for working at the hackathon: Institute of Physiology named after I.P. Pavlova, Institute of Cytology RAS, St. Petersburg State University, Federal Scientific Center for Physical Medicine, JetBrains BioLabs, Protein Institute RAS, Genotek, MIPT, iBinom and others. </font></font><br>
 <br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In 2019, the Garlic team took the main prize of 150,000 rubles. For 48 hours allocated to work, the team created a tool that allows you to search for genomic rearrangements of a given structure. We asked the curator of the project, Dmitry Konanov, to talk about the project, the hackathon and, in general, the life of bioinformatics.</font></font><br>
<br>
<img src="https://habrastorage.org/webt/wn/on/rk/wnonrkejub3cakiy8camajib4b4.jpeg"><br>
<a name="habracut"></a><br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- Tell me, what were you doing at that moment when you were a member of the hackathon?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- At the time of my participation in the hackathon, I worked in the bioinformatics laboratory at the Federal Scientific and Practical Center of Physical-Chemical Medicine of the FMBA of Russia (Federal Scientific and Clinical Center for Physico-Chemical Medicine of the Federal Medical and Biological Agency), wrote a diploma there. Now I continue to work in the center. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- Why did you decide to participate in BioHack?</font></font></b> <br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- Somehow it happened so spontaneously. The deadline was already approaching - one of the last days of receiving projects was on, the laboratory asked me if I wanted to participate in the hackathon: I just had to send the project. I wrote something in about 15 minutes and sent a request. </font></font><br>
<br>
<img src="https://habrastorage.org/webt/dz/ne/e6/dznee6w9p7p9e7l-omvcykkqke8.jpeg"> <br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- So it was a project that you already worked on in the laboratory?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
- I wanted to work on it, I started working on it, but it was very unfinished. At the hackathon, we brought it to the state I wanted to bring to - the algorithm has become more automated. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- Tell me how the idea of ‚Äã‚Äãthe project came about.</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- In general, the initial idea is not mine, but Alexander Manolov, Ph.D. bioinformatics laboratories. This is the person who was at that time the head of my diploma.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It is known that the genomes of bacteria are very plastic. Many events can occur in them: the transfer of genes from one bacterium to another, a change in their sequence, insertion and removal of fragments of the genome. The idea is this: let there be 4 bacterial genomes. Each genome consists of 5 genes: the first XYZTF genome, the second XRLAF genome, the third XYKTF genome, and the fourth XYLTF. (Figure ‚ÄúPerestroika in the graphs‚Äù). In our example, the same letters in the genomes correspond to homologous (one might say, the same) genes. The sequence of letters shows the sequence of genes in the genomes. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
We define each gene from the sample of genomes as a node of the graph and draw edges between those gene-nodes that are located sequentially in at least one of the genomes of the sample.</font></font><br>
<br>
<img src="https://habrastorage.org/webt/wa/t_/-h/wat_-h6ts1bqkmyhcee9qiaqs5a.png"><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Thus, we obtain a graph that contains information on all possible variants of the alternation of genes in the selected genomes. And with this graph structure you can already do whatever your heart desires. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Our first task was to look for regions with a high local involvement in graphs that arise in the so-called hot spots of genomic rearrangements - places where, for reasons that are not always understood, the genome changes intensively from strain to strain. We called the measure of entanglement in the vicinity of the node genome complexity. This value essentially numerically shows how often perestroika occurs in certain regions. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- And what is the essence of the problem that you solved with the team at the hackathon?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- At the hackathon we brought out a task more mathematically beautiful or something.</font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Any genome rearrangement, be it a deletion (approx. Loss of a chromosome site), insertion or inversion (approx. Reverse order of genes of a chromosome site), leads to the formation of subgraphs of a certain topology in our large graph of rearrangements. And I thought it would be good if we were able to look for specific subgraphs corresponding in structure to the perestroika of interest to us. This would make it possible to efficiently find points in the genome at which events occur more often, and to compare the frequency between different species and genera of bacteria. It is known, for example, that there are parts of the genome that are forbidden for inversions, and areas where inversions occur most often.</font></font><br>
 <br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
A purely fundamental interest was to look at the frequency distribution profiles along the genome and for other types of rearrangements. If we talk about practical significance, then this story is directly related to biotechnology: we think that, knowing the susceptibility of different fragments of the genome to inserts, we can assume in advance where it is more likely that a random fragment of foreign DNA will be inserted. But we did not check it.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Even before the hackathon, I handled something with pens, wrote a rather crooked algorithm that would look for a specific pattern (code name Smile, due to its characteristic appearance). I found the frequency and distribution along the genome for many species, even some funny things were found out, for example, in bacteria with a large number of smileys, rearrangements of any type occurred equally likely along the entire genome, and in bacteria with a small number of smileys only in a limited number of hot spots (at close integral in genome frequency). Of course, I wanted to do something more universal so that I could ask any possible subgraphs for the search. I brought this idea to the hackathon.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As a result of two days of work, we got a tool called GARLIC-Finder - a tool for studying genomic rearrangements of a given structure. We wrote a universal language for specifying subgraphs for searching, but since such a task is NP-complicated, looking ‚Äúhead-on‚Äù turned out to be only small static subgraphs. Therefore, we added the ability for the user to add custom algorithms that are optimized for the search for specific patterns. At the hackathon, we settled on three patterns - a pair of genes between which inserts (Garlic), transpositions of a genome fragment (Penguin) and a gene with a very rich environment (Spider) often occur (Fig. ‚ÄúSearch for subgraph-rearrangements‚Äù). Garlic was the first and therefore gave the name to our Tulu. It has become an acronym: </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">G</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> enome re </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">AR</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> rangements</font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">L</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> earning </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">I</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> nterfa </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">C</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> e. </font></font><br>
<br>
<img src="https://habrastorage.org/webt/u_/qh/eg/u_qhegi5xqeb_mml4suuy3mzazu.png"><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
I even took advantage of this a bit later. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- Little? That is, this project did not advance further than the hackathon?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- Now the problem is that we are still at the stage of publishing a large article on graphs and on genomic complexity. A person who writes a dissertation on this topic is engaged in this. We sent the first option in the summer, but it was rejected, unfortunately. The other day sent again, already in another edition. If all goes well, perhaps we will continue to dig in that direction. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- What did the hackathon give you?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- The project has become a big part of my thesis. New optimization ideas have come up. Well, in general, I myself learned a lot of new things.</font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- What did you spend money on, if not secret?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- It's no secret, a good player with headphones :). </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- What programming language was used to solve the problem?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- Python, exclusively Python. And different libraries to it. To work with NetworkX graphs, to visualize Graphviz and its binding to a Python. Well, the classic Matplotlib and Pandas for working with data. And one self-written library is </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">gene-graph-lib</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- And who was on your team?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- Two programmers and one biologist. Everyone turned out to be very helpful. </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
What was your global goal, why did you decide to send the project to a hackathon?</font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
I wanted to solve a problem and solve it effectively. I planned to do it myself, but here a unique opportunity came up, and I decided to use it. Well, I just wanted to see what a hackathon is. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- Do you like it?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- Wonderful, just wonderful! The organization, the food, the room where all this happened, the people are wonderful. There was nothing to complain about at all. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It would be very good if they allowed me to use local monitors, staff equipment, as I understand it - the hackathon was held in the EPAM office - but, of course, we were not allowed. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- How did you prepare for the hackathon? What needs to be done, in addition to taking your equipment?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- To the leaders </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">(approx. Curators)</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">there was a requirement to prepare a presentation for 1.5-2 minutes about the project. It is important for the participants to carefully read the terms of the projects in order to see what requirements the leaders have for the team members. It may happen that a person on a laptop is fully equipped with the environment of the second Python, and the curator, for example, the third. This does not matter, but it may take extra time to reinstall the environment, and you just had to carefully read that you need a third Python. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But in terms of knowledge of how to prepare, not everything is clear here. Naturally, you need to be able to program in the required languages ‚Äã‚Äãand fumble a bit in the context of the problem that is proposed as a project. Although we had a team member who did not know biology at all, but was very useful - it was he who wrote the language parser for defining subgraphs, this task fell entirely on his shoulders.</font></font><br>
 <br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- You already spoke about the organization, the premises, the food. Where did you sleep? And did you sleep at all?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
‚ÄúFor 48 hours, I slept for four hours, I guess.‚Äù I was always on the site, on the last night I just left for the hotel. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- That is, the participants need to be mentally prepared for this.</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- And morally and physically especially. If a person has experience in preparing for some terrible sessions, when you do not sleep for two nights, this is a good preparation. I had such cases during my studies, so I was ready. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- What is your global goal? Why do you do bioinformatics?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
- In general, I accidentally came into bioinformatics. I studied at the faculty of the Academy of Agricultural Sciences of RAS. There, students, starting from the second year, are required to go to scientific work one day a week at one of the institutes of the Russian Academy of Sciences. I responded to the proposal of the IBCh RAS, without any idea what I would have to do. I came there and it turned out that I would do an analysis of NGS data and proteomics. Then I started to learn Python and understand bioinformatics. He worked there for two years, the project seemed to stall a bit, and I went to where I am working now. </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
I like it. I always loved both mathematics and biology, somehow it happened. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- What books, courses, lectures, films do you recommend for children to watch?</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
- There is a course on bioinformatic algorithms at Coursera from the University of San Diego, in the creation of which I participated</font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Pavel Pevzner</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , on </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Stepic</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> he also is. I solved some problems there - quite useful. They allow you to pump knowledge both in molecular biology and in coding. The essence of most tasks is that you need to program some kind of simple sequence analysis or the like. I know that the Institute of Bioinformatics conducts guest </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">lectures</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> that can be viewed on YouTube, in addition, they have courses on </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Stepic</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . In Python, I faithfully read about 500 of the first pages of Learning Python by </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Mark Lutz</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , and then just reading the documentation, changelogs and practice. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The most important thing is to solve problems. It‚Äôs useless to read the theory, and in the process of solving problems you learn to solve real problems.</font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- Do you plan to participate in the hackathon this year? </font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
‚ÄúYes, I think so.‚Äù </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- With what? </font><font style="vertical-align: inherit;">Or is it still a secret? </font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
- There are two options, while ideas are being formed. </font><font style="vertical-align: inherit;">I will not voice it. </font><font style="vertical-align: inherit;">I still have a whole month. </font><font style="vertical-align: inherit;">I‚Äôll give it, probably, as always at the last moment :) </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- And what is now being discussed in the world of bioinformatics? </font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
- Often love hype themes. </font><font style="vertical-align: inherit;">I have a student from the Russian Chemical Technical University who writes a diploma on a graph topic, so he decided to build a graph on the recently published genome of the coronavirus and its relatives. </font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- Intrigued. </font><font style="vertical-align: inherit;">We will wait for new discoveries and new interesting projects from you and your colleagues! </font></font></b><br>
<br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">You can </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">submit the</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> project until February 28, and register as a participant until March 5 at </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;">biohack.ru</font></a><font style="vertical-align: inherit;"> .</font></font><br>
</i></div>
      
    </div>
<section class="more-articles-navigation-panel js-more-articles-navigation-panel">
<h4>More articles:</h4>
<nav class="list-of-articles-container js-list-of-articles-container"><ul class="list-of-pages js-list-of-pages">
<li><a href="../en488020/index.html">Vivaldi 2.11 - One Touch Beauty</a></li>
<li><a href="../en488024/index.html">Fantastic advisory locks and where they live</a></li>
<li><a href="../en488026/index.html">Jewelery, Formlabs 3D Printer and Environmental Care</a></li>
<li><a href="../en488028/index.html">How to start making a game in UE4</a></li>
<li><a href="../en488030/index.html">How to organize skins in symfony</a></li>
<li><a href="../en488036/index.html">The domain corp.com is for sale. It is dangerous for hundreds of thousands of corporate computers running Windows</a></li>
<li><a href="../en488038/index.html">Surprise Basket - Save Our Push</a></li>
<li><a href="../en488042/index.html">10/50/99: how to give feedback</a></li>
<li><a href="../en488046/index.html">The book "Development using quantum computers"</a></li>
<li><a href="../en488048/index.html">To illuminate with a searchlight. How to get maximum impact on "social" hackathons?</a></li>
</ul></nav>
</section><br />
<a href="../../allArticles.html"><strong>All Articles</strong></a>
<script src="../../js/main.js"></script>

<!-- Yandex.Metrika counter -->
<script type="text/javascript" >
  (function (d, w, c) {
      (w[c] = w[c] || []).push(function() {
          try {
              w.yaCounter63335242 = new Ya.Metrika({
                  id:63335242,
                  clickmap:true,
                  trackLinks:true,
                  accurateTrackBounce:true,
                  webvisor:true
              });
          } catch(e) { }
      });

      var n = d.getElementsByTagName("script")[0],
          s = d.createElement("script"),
          f = function () { n.parentNode.insertBefore(s, n); };
      s.type = "text/javascript";
      s.async = true;
      s.src = "https://mc.yandex.ru/metrika/watch.js";

      if (w.opera == "[object Opera]") {
          d.addEventListener("DOMContentLoaded", f, false);
      } else { f(); }
  })(document, window, "yandex_metrika_callbacks");
</script>
<noscript><div><img src="https://mc.yandex.ru/watch/63335242" style="position:absolute; left:-9999px;" alt="" /></div></noscript>

<!-- Google Analytics -->
  <script>
    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
    ga('create', 'UA-134228602-13', 'auto'); ga('send', 'pageview')
  </script>
  <script src="https://www.google-analytics.com/analytics.js" async defer></script>

</section>

<footer class="page-footer">
  <div class="page-footer-legal-info-container page-footer-element">
    <p>
      Geek Week | <span class="page-footer-legal-info-year js-page-footer-legal-info-year">2020</span>
    </p>
  </div>
  <div class="page-footer-counters-container page-footer-element">
    <a class="page-footer-counter-clustrmap" href='#'  title='Visit tracker'><img src='https://clustrmaps.com/map_v2.png?cl=698e5a&w=271&t=t&d=i62cJ2037o_BACd40gCrIso3niu0Sjx2sDFYJkeYdRk&co=3a3a3a&ct=ffffff'/></a>
  </div>
</footer>
  
</body>

</html>