<!doctype html>
<html class="no-js" lang="en">

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-134228602-13"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-134228602-13');
  </script>

  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>‚ôÇÔ∏è üò® üïû Sars miraculous? Genealogy of Wuhan Coronavirus üç∑ ü¶Ç üõåüèø</title>
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <meta name="description" content="No, well, what is man-made? What kind of nonsense? I thought when I first heard the hypothesis that Kovid-19 was caused either by laboratory leakage, ...">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <link rel="stylesheet" href="../../css/main.css">

  <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">

  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script>window.jQuery || document.write('<script src="../../js/vendors/jquery-3.3.1.min.js"><\/script>')</script>

  <script>document.write('<script src="//pagea' + 'd2.googles' + 'yndication.com/pagea' + 'd/js/a' + 'dsby' + 'google.js"><\/script>')</script>
  <script>
        var superSpecialObject = {};
        superSpecialObject['google_a' + 'd_client'] = 'ca-p' + 'ub-6974184241884155';
        superSpecialObject['enable_page_level_a' + 'ds'] = true;
       (window['a' + 'dsbygoogle'] = window['a' + 'dsbygoogle'] || []).push(superSpecialObject);
  </script>
</head>

<body>
  <!--[if lte IE 9]>
    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
  <![endif]-->
  <header class="page-header js-page-header">
    <a class="page-header-logo-container" href="https://geek-week.github.io/index.html"></a>
    <div class="page-header-text">Get best of the week</div>
  </header>
  <section class="page js-page"><h1>Sars miraculous? Genealogy of Wuhan Coronavirus</h1><div class="post__body post__body_full">
      <div class="post__text post__text-html post__text_v1" id="post-content-body" data-io-article-url="https://habr.com/ru/post/497956/"><div style="text-align:center;"><img width="450" src="https://habrastorage.org/getpro/habr/post_images/f61/9f6/4c6/f619f64c632d1eac215da29ae752b100.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
No, well, what is man-made? What kind of nonsense? I thought when I first heard the hypothesis that Kovid-19 was caused either by laboratory leakage, or even by a targeted bio-attack. And each time he simply dismissed these speculations when they once again swam up to me in a stormy stream of coronavirus infoshum. Well, think of it, there is a virology institute in Wuhan, you never know. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
At some point, it was already necessary to dismiss it reasonably, because supporters of man-madeness </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">began to substantiate</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> their theses on the possible artificial nature of the virus with arguments from molecular biology, and here I already wanted to smash their conspiracy with cold scientific facts. Well, if not as </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">authors of an article in Nature</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (it seemed to me), then at least as </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Panchin</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> respected by me </font><font style="vertical-align: inherit;">.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
And here, in pursuit of the arguments against the man-made virus, the virus of doubt infected me. What, in fact, is the reason for doubt? The fact that the deeper you get into the activities of coronavirusologists over the past 15‚Äì20 years, the better you understand that creating exactly such chimeras like CoV2 was commonplace in them. </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">And CoV2 is an obvious chimera, based on the bat strain of RaTG13, in which the receptor binding site (RBM) in the spike protein is replaced from the bat by pangolinium, and in addition, a special section of 4 amino acids is inserted, which creates a furin cleavage site, which, like virologists have previously found that significantly expands the "repertoire" of the virus in terms of in whose cells it can penetrate.</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Most likely, it was thanks to this new furin site that the new mutant managed to jump from the original carriers to people. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Taking into account the heights that genetic engineering has reached today, synthesizing CoV2 using the method described above would not be difficult even for a novice specialist. </font><font style="vertical-align: inherit;">Indeed, virologists, including </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Shi Zhengli</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , the </font><font style="vertical-align: inherit;">head of the coronavirus </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">department</font></a><font style="vertical-align: inherit;"> at the Wuhan Institute of Virology </font><font style="vertical-align: inherit;">, have repeatedly been involved in such things - as replacing RBM in one type of virus with RBM from another</font></font><a name="habracut"></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">(here is the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">work</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> of the Shi Zhengli group from 2007), and by adding a new furin site that can give the coronavirus specific to one animal species the opportunity to start using the ACE2 receptor of other species.</font></font><br>
<br>
<blockquote><img src="https://habrastorage.org/webt/fi/0x/4z/fi0x4z2nhcc3-3bdzugakb4csji.png" align="left"><h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">UFO Care Minute</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The pandemic COVID-19, a potentially severe acute respiratory infection caused by the SARS-CoV-2 coronavirus (2019-nCoV), has officially been announced in the world. </font><font style="vertical-align: inherit;">There is a lot of information on Habr√© on this topic - always remember that it can be both reliable / useful, and vice versa.</font></font><br>
<br>
<h4><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">We urge you to be critical of any published information.</font></font></h4><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Official sources</font></font></b>
                        <div class="spoiler_text"><ul>
<li><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow">C   </a></li>
<li><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow">C </a></li>
<li><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow">  ()</a></li>
<li><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"> </a></li>
<li>       </li>
</ul><br>
     ,      .</div>
                    </div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Wash your hands, take care of your loved ones, stay at home whenever possible and work remotely. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Read publications about: </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">coronavirus</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> | </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">remote work</font></font></a></blockquote><br>
<div style="text-align:center;"><img width="800" src="https://habrastorage.org/getpro/habr/post_images/5df/3d5/aad/5df3d5aad6246e32b038f36802afc6ff.png"></div> <blockquote><i><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Shi Zhengli in his laboratory at the Wuhan Institute of Virology</font></font></b></i></blockquote><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But before breeding conspiracy here, let's delve into the </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">bio</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ogy.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Biology</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
So, let's start from the stove. </font><font style="vertical-align: inherit;">What kind of furin site, what kind of RBM, what kind of spike protein in general? </font><font style="vertical-align: inherit;">In fact, if you wade through the jungle of terminology, then everything is conceptually simple. </font><font style="vertical-align: inherit;">For example, a spiky protein is the very thing sticking out of a virus particle (S protein), for which these viruses ‚Äúcrowned‚Äù:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/7e9/8ea/1a0/7e98ea1a0c6c81da5ae9a3aa366cbbe9.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It is with the help of these proteins that the virion clings to the receptor of the victim cell (ACE2 in our case), then to penetrate inside. </font><font style="vertical-align: inherit;">Therefore, we can say that this is the most important part of the virus, because it determines which animals can affect and which ones are not - ACE2 receptors in different species are slightly structurally different. </font><font style="vertical-align: inherit;">At the same time, out of the entire huge 30 kilobase genome by viral standards, the gene of this protein is only 12‚Äì13%, that is, about 1300 amino acids. </font><font style="vertical-align: inherit;">This is how the spike protein is structured in CoV2 and its close relatives:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/9e4/1ca/2e0/9e41ca2e0b093ee7558eb089d38bac13.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As can be seen from the figure above, the S protein consists of two subunits: S1 and S2. It is S1 that interacts with the ACE2 receptor, and the place it does is called the Receptor Binding Domain (RBD), and the area of ‚Äã‚Äãdirect contact, the holy of holies, is called Receptor Binding Motif (RBM). Here is a beautiful illustration from an equally beautiful </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">work</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> :</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/046/388/e7f/046388e7fb0beea71b7be894ff6a6cba.png"></div> <blockquote><i><b>  RBD CoV2,   ACE2.<br>
</b>()      2019-nCov. NTD, N- . RBD, - . RBM, - . SD1,  1. SD2,  2. FP,  . HR1,   1. HR2,   2. TM,  . IC,  .<br>
()     RBD 2019-nCov. RBM    .<br>
()   RBD 2019-nCov,   ACE2. ACE2    .  RBD 2019-nCoV    ,  RBM ‚Äî  .    RBD 2019-nCov        . N-  ACE2,   , .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text"><b>Overall structure of 2019-nCoV RBD bound with ACE2.<br>
</b>(a) Overall topology of 2019-nCoV spike monomer. NTD, N-terminal domain. RBD, receptor-binding domain. RBM, receptor-binding motif. SD1, subdomain 1. SD2, subdomain 2. FP, fusion peptide. HR1, heptad repeat 1. HR2, heptad repeat 2. TM, transmembrane region. IC, intracellular domain. <br>
(b) Sequence and secondary structures of 2019-nCoV RBD. The RBM is colored red. <br>
() Overall structure of 2019-nCoV RBD bound with ACE2. ACE2 is colored green. 2019-nCoV RBD core is colored cyan and RBM is colored red. Disulfide bonds in the 2019-nCoV RBD are shown as stick and indicated by yellow arrows. The N-terminal helix of ACE2 responsible for binding is labeled.</div>
                    </div></i></blockquote><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
So here. When the CoV2 genome was only decrypted, at first no strains directly related to it were known. But already on January 23, 2020, Shi Zhengli </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">released a work</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in which she announced that CoV2 is 96% coincident with the RaTG13 strain, which her laboratory in 2013 isolated from Yunnan bats. True, outside her laboratory until January 2020, no one knew about this strain. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It was immediately clear that RaTG13 is a special boy. Take a look at the chart:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/c26/8e0/b28/c268e0b28a933cb678eb4bd6a1cf8648.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
This is a graph of the similarity of CoV2 to other known strains. </font><font style="vertical-align: inherit;">The higher the curve, the higher the percentage of nucleotide matching. </font><font style="vertical-align: inherit;">As you can see, in the region of the gene of the very spiky protein (S), only RaTG13 is more or less close to CoV2, and all other strains in this place go to the peak - both virus strains from other bats and the first SARS-CoV (red curve ) </font><font style="vertical-align: inherit;">But so far there is nothing suspicious - are there any unknown strains of Yunnan caves still harbor unknown strains of science? </font><font style="vertical-align: inherit;">Well, yes, it‚Äôs not very clear how exactly the virus got to Wuhan from there, but what doesn‚Äôt happen.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Pangolins</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Then pangolins appeared on the scene: in February, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">another group of Chinese scientists discovered</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in their bins a strain of pangolin coronavirus, which, although generally worse than RaTG13, was similar to CoV2 (90%), in that the RBM spike protein was almost identical - it was different only for 1 amino acid (see the two upper sequences, dots mean coincidence with the first sequence):</font></font><br>
<br>
<div style="text-align:center;"><img width="500" src="https://habrastorage.org/getpro/habr/post_images/c58/842/552/c588425528044baf081f05588fffca22.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Moreover, in the first quarter of the S protein, the pangolinium strain is not similar to CoV2, and the sequence after RBM in all three strains (CoV2, Pangolin, RaTG13) more or less coincides. </font><font style="vertical-align: inherit;">But RaTG13‚Äôs RBM itself is very different from CoV2, which can be seen from the steep dip of the green RaTG13 graph compared to the red CoV2 graph in the RBM region (pink vertical bar) in the following graph:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/ac7/c6f/ef2/ac7c6fef2883f8b4bf72d5578990ec76.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
This difference is also confirmed by the phylogenetic analysis of these three areas highlighted in the graph above - according to RBM, the pangolin strain is closer to CoV2 than RaTG13, but on the left and right of RBM to CoV2 it is closer to RaVG13. </font><font style="vertical-align: inherit;">That is, there is obvious recombination, as the authors themselves and some other works mention. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, where did the pangolins come from the researchers? </font><font style="vertical-align: inherit;">And from here:</font></font><br>
<br>
<div style="text-align:center;"><img width="400" src="https://habrastorage.org/getpro/habr/post_images/07d/a75/417/07da75417ef55db38945c9b0f13819d1.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
They were confiscated from smugglers by Chinese customs and transferred to a rehabilitation center in Guangdong, where they died with severe coronavirus symptoms. </font><font style="vertical-align: inherit;">This, of course, could not but interest local virologists, who isolated different biomaterials from them:</font></font><br>
<blockquote><i>,   ,           - 2019 .        (<i>Manis pentadactyla</i>)  25   (<i>Manis javanica</i>).         ;       ,     ,       .      ,  , , , ,         ,     .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Pangolins used in the study were confiscated by Customs and Department of Forestry of Guangdong Province in March-December 2019. They include four Chinese pangolins (Manis pentadactyla) and 25 Malayan pangolins (Manis javanica). These animals were sent to the wildlife rescue center, and were mostly inactive and sobbing, and eventually died in custody despite exhausting rescue efforts. Tissue samples were taken from the lung, lymph nodes, liver, spleen, muscle, kidney, and other tissues from pangolins that had just died for histopathological and virological examinations.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">By the way, and not only local, because other Chinese researchers (Hong Kong in this case) also received samples of confiscated pangolins and in February 2020 also </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">released a similar work</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , noting clear signs of recombination in the CoV2 spike protein:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">We received samples of frozen tissues (lungs, intestines, blood) that were taken from 18 Malay pangolins ( </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Manis javanica</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ) during August 2017 - January 2018. </font><font style="vertical-align: inherit;">These pangolins were obtained during anti-smuggling operations by Guangxi Customs. </font><font style="vertical-align: inherit;">It is amazing that high-performance sequencing of their RNA revealed the presence of coronaviruses in six (two lungs, two intestines, one mixture of lungs and intestines, one blood) of 43 samples. </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
...</font></font><br>
 , ,        ,    RaTG13  2019-CoV2  (. 1c, d).  , 2019-CoV2             -  (RBD;   97,4%;     . 1c  . 2a),       2019-CoV2    RaTG13.     RaTG13  2019-CoV2   89,2%    RBD.      2019-CoV2        RBD,   RaTG13       2019-CoV2    ( 442,  SARS-CoV ).<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">We received frozen tissue (lungs, intestine, blood) samples that were collected from 18 Malayan pangolins (<i>Manis javanica</i>) during August 2017-January 2018. These pangolins were obtained during the anti-smuggling operations by Guangxi Customs. Strikingly, high-throughput sequencing of their RNA revealed the presence of coronaviruses in six (two lung, two intestine, one lung-intestine mix, one blood) of 43 samples. With the sequence read data, and by filling gaps with amplicon sequencing, we were able to obtain six full or nearly full genome sequences ‚Äî denoted GX/P1E, GX/P2V, GX/P3B, GX/P4L, GX/P5E and GX/P5L ‚Äî that fall into the 2019-CoV2 lineage (within the genus Betacoronavirus) in a phylogenetic analysis (Figure 1a).<br>
‚Ä¶<br>
More notable, however, was the observation of putative recombination signals between the pangolins coronaviruses, bat coronaviruses RaTG13, and human 2019-CoV2 (Figure 1c, d). In particular, 2019-CoV2 exhibits very high sequence similarity to the Guangdong pangolin coronaviruses in the receptor-binding domain (RBD; 97.4% amino acid similarity; indicated by red arrow in Figure 1c and Figure 2a), even though it is most closely related to bat coronavirus RaTG13 in the remainder of the viral genome. Bat CoV RaTG and the human 2019-CoV2 have only 89.2% amino acid similarity in RBD. Indeed, the Guangdong pangolin coronaviruses and 2019-CoV2 possess identical amino acids at the five critical residues of the RBD, whereas RaTG13 only shares one amino acid with 2019-CoV2 (residue 442, human SARS-CoV numbering).</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">By the way, the authors of this article also highlighted the explicit phylogenetic mosaicity of the CoV2 spike protein:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Interestingly, the phylogenetic analysis of only synonymous sites in RBD showed that the phylogenetic position of the guangdong pangolin is consistent with that of the rest of the viral genome, namely that it is not the closest relative of 2019-CoV2 (Figure 2b). </font><font style="vertical-align: inherit;">Therefore, it is possible that the similarity of amino acids in the RBD of pangolin coronaviruses and 2019-CoV2 is due to selectively mediated convergent evolution rather than recombination, although it is difficult to choose between these scenarios based on the available data.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Interestingly, a phylogenetic analysis of synonymous sites alone in the RBD revealed that the phylogenetic position of the Guangdong pangolin is consistent with that in the remainder of the viral genome, rather than being the closest relative of 2019-CoV2 (Figure 2b). Hence, it is possible that the amino acid similarity between the RBD of the Guangdong pangolin coronaviruses and 2019-CoV2 is due to selectively-mediated convergent evolution rather than recombination, although it is difficult to choose between these scenarios on current data.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Translated from scientific, the authors' words mean that if we analyze the entire RBD of the three strains, discarding the obvious differences (non-synonymous substitutions) between them, which are mainly attributable to RB </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">M</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (which, I recall, is identical between CoV2 and Pangolin), and build phylogenetic tree by synonymous substitutions, CoV2 is nevertheless closer to RaTG13, and not to the pangolin strain. Which is rather strange in the light of the fact that the pangoliniy with CoV2 has identical RBM (that is, a segment inside RBD).</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The authors further theorize that this may be the result of convergent evolution, that is, in other words, that strains of CoV2 and pangolins came to the identical RBM each in their own way, and not through joint recombination of common ancestors. </font><font style="vertical-align: inherit;">Because it was really very strange that recombination had to happen - as if someone just took a piece of RBM from a pangolin strain and replaced it with RBM in RaTG13. </font><font style="vertical-align: inherit;">And this is not like evolution, but, forgive Darwin, Intelligent Design.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Genealogy of Crowned Persons</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In order to better understand the origin of CoV2, let's take another look at the sequence of the spike protein in our trinity: CoV2, RaTG13 and pangolin - compare the pairwise difference between them (identical amino acids are marked with dots, red letters indicate the difference, and dashes indicate deleted / added amino acids) :</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/wp/_2/g9/wp_2g9z0flpa3oxfojossayyf9o.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It can be seen with the naked eye that in the first quarter of the sequence, the pangolinium strain is far from CoV2 and RaTG13. Well, RaTG13, if not for the plot in the RBM region (red rectangle), would be sooo close to CoV2. But, as I already said, the same site in CoV2 is closest to the pangolin strain. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, what about other pangolin strains? Let's get a look. After all, so far we have only analyzed the virus isolated from pangolins confiscated by customs in 2019. And there was also a batch of pangolins confiscated in 2017, and they also had a similar strain isolated. If we compare RaTG13 with the genomes of viruses from the pangolins of 2017 and 2019, then everything is also interesting here:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/a6/gg/zp/a6ggzpvuxowphdftv_d6l61n0k8.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the first quarter of the S protein, the pangolin strains from 2017 are closer to RaTG13 (and CoV2) than their pangolin counterpart from 2019 (MP789). At the same time, all three have a clear recent common ancestor in the areas highlighted by green rectangles, and in these areas RaTG13 and pangolin-19 (MP789) are closer to it than pangolin-17, since it has several common mutations with RaTG13 (circled in red and blue ellipses), which are not observed in the legend-17. At the same time, the RBM for all three is different and different in approximately the same proportion, and in similar places.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Perhaps, even after the ancestors of RaTG13 and MP789 separated, MP789 replaced a large portion in the first quarter of S protein (which did not occur in RaTG13 and pangolin-17), and the rest of the S protein remained common for all three. And then the paths of the RaTG13 and MP789 gene pools came together again and in a fit of passion gave birth to CoV2. It is also possible that the ancestor of RaTG13 is the result of recombination of the ancestors of the pangolin strains.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It is also interesting to see the rather unique identical mutation (QTQTNS) in RaTG13 and pangolin-19 right in front of the place where CoV2 has a new furin cleavage site, which arose due to the insertion of 4 new amino acids in this place (PRRA). </font><font style="vertical-align: inherit;">If you look at the nucleotide sequence around this identical mutation, you can see that RaTG13 and CoV2 are closer to each other than to pangolin-19, since they managed to accumulate several common mutations (highlighted in blue):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/da1/ff2/051/da1ff2051751e4f50cc5baeb856bf638.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, with Orf1ab, CoV2 also has a phylogenetic leapfrog: 1a is closer to RaTG13, but 1b is closer to pangolin-19:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/047/a72/4ab/047a724ab92edcc97b2b7866a3ba79a4.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
That is, it turns out that the ancestor of CoV2 sinned with the common ancestor of pangolin-19, at least twice? </font><font style="vertical-align: inherit;">For the first time - when he (along with a common ancestor with RaTG13) inherited Orf1ab and the second part of the spike-like protein with the QTQTNS mutation, and secondly - when he acquired 1b with RBM, which is already different from RaTG13-s. </font><font style="vertical-align: inherit;">No, of course, this is possible and in itself is not particularly noteworthy - after all, these viruses mutate and recombine constantly. </font><font style="vertical-align: inherit;">Another question is where exactly the bats and pangolin viruses can be found with each other for such orgies - in mountain caves, "wet markets", shelters for confiscated animals, or even in laboratories. </font><font style="vertical-align: inherit;">But let's take a moment with these questions. </font><font style="vertical-align: inherit;">First, consider the most eye-catching aspect of the new virus - a 4-amino acid sidebar that turned it into a super-killer.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">I‚Äôm hitting neatly, but hard</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It is completely impossible to ignore this PRRA box between S1 and S2. As a splinter it sticks out in the genome of our new CoV2. This box is not simple, but golden. She creates the very </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">furin cleavage site</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , which I mentioned at the very beginning. What kind of beast is this? I will try to explain briefly. Remember the structure of our spike protein? Here is a visual diagram:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/52a/f0d/c3c/52af0dc3cfa126b2bb6c991ddc451d33.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The protein consists of two parts, S1 and S2, of which S1 is responsible for primary contact with the receptor (the same Receptor Binding Domain / Motif), and S2 is responsible for fusion with the cell membrane and penetration into the cell. It starts the fusion process marked with a yellow fusion peptide, but in order for it to start its dirty business, someone must cut the S protein in one of the sites highlighted by rhombuses in the diagram above. The virus does not have its own ‚Äúcutters‚Äù (there are </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">others</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">, but this is irrelevant), so he relies on various proteases of his victims, the benefit of such proteases, as you can understand by the abundance of colors of those same rhombuses, there are several types. But not all of them are equal, and not all types of cells have the proteases needed by the virus. And furin is just one of the most effective, and even it lives not only on the surface of cells, but also inside. Most clearly, the danger of the new furin site is demonstrated by the difference between CoV2 and its ‚Äúforerunner‚Äù SARS-CoV:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/595/45b/ada/59545bada2129fc33ffe0c06d0ba6b1b.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As can be seen from the diagram, in the case of CoV2, thanks to the furin site, it is not two, but three classes of proteases (three multi-colored pacmen) that can cut its S protein outside the cell. But perhaps the most important difference is that furin is also present inside the cell, so it can cut the S protein immediately after assembly of the virion, thereby increasing the ability of new virions to merge with other cells - without leaving the box office, so to speak. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, it </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">is likely</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> that it is the new furin site that plays an important role in the pronounced age-dependent morbidity and mortality of CoV2:</font></font><br>
<blockquote><i>  ,   , ,   ,  ,              SARS-CoV-2.         ()    COVID-19    .             SARS-CoV-2 ,      .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Patients with hypertension, diabetes, coronary heart disease, cerebrovascular illness, chronic obstructive pulmonary disease, and kidney dysfunction have worse clinical outcomes when infected with SARS-CoV-2, for unknown reasons. The purpose of this review is to summarize the evidence for the existence of elevated plasmin(ogen) in COVID-19 patients with these comorbid conditions. Plasmin, and other proteases, may cleave a newly inserted furin site in the S protein of SARS-CoV-2, extracellularly, which increases its infectivity and virulence.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Oh yes, it cuts furin proteins in strictly defined places, namely after the RxxR sequence (that is, Arg-XX-Arg, where X can be any amino acid). Moreover, if arginine is also in the second or third place (that is, RRxR or RxRR), then the cleavage efficiency of this site increases significantly. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Therefore, the appearance of a new furin cleavage site by specialists was </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">immediately noticed</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . Still would! After all, none of the closest or even distant relatives of Cov2 has such a site - those coronaviruses that have it are </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">only 40%</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> close to Cov2 in terms of the genome:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">It was found that all spike proteins with a SARS-CoV-2 protein homology greater than 40% did not have a furin cleavage site (Figure 1, Table 1), including Bat-CoV RaTG13 and SARS-CoV (with sequence identity as 97.4 % and 78.6%, respectively). </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">The RRAR furin site in SARS-CoV-2 is unique in its family thanks to the unique PRRA insert. </font><font style="vertical-align: inherit;">This site in SARS-CoV-2 could hardly have evolved from MERS, HCoV-HKU1, etc. </font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">From the currently available sequences in the databases, it is difficult for us to find the source. </font><font style="vertical-align: inherit;">Perhaps there are many more evolutionary intermediate sequences awaiting discovery.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">It was found that all Spike with a SARS-CoV-2 Spike sequence homology greater than 40% did not have a furin cleavage site (Figure 1, Table 1), including Bat-CoV RaTG13 and SARS-CoV (with sequence identity as 97.4% and 78.6%, respectively). <b>The furin cleavage site ‚ÄúRRAR‚Äù in SARS-CoV-2 is unique in its family, rendering by its unique insert of ‚ÄúPRRA‚Äù. The furin cleavage site of SARS-CoV-2 is unlikely to have evolved from MERS, HCoV-HKU1, and so on.</b> From the currently available sequences in databases, it is difficult for us to find the source. Perhaps there are still many evolutionary intermediate sequences waiting to be discovered.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Here is a clear illustration from the same article as the above quote (coronaviruses with a furin site are marked in pink, 3 different strains of Cov2 are shown at 10 o‚Äôclock):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/09a/763/dd8/09a763dd85fea5d0fe7d3ef8f11ae239.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The closest relative with the furin site is the HKU5 strain, isolated by the Shi Zhengli team in 2014 in Guangzhou from bats of the genus </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Pipistrellus</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (added to </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">GenBank</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in 2018). But he is a very distant relative - their spike-like proteins coincide only by 36%. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In general, scientists are confused. Where did this 12 nucleotide insert come from? Could it be man-made? Quite. Indeed, virologists have dealt with such inserts repeatedly, and since olden times. For example, Americans inserted RRSRR into the spike protein of the first SARS-CoV </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">back in 2006</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> :</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">To investigate whether proteolytic cleavage at the sequence of the main amino acid residues could contribute to cell fusion activity, we mutated the spike-like SARS-CoV protein to create a furin recognition site (RRSRR) in one of two places.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">To investigate whether proteolytic cleavage at the basic amino acid residues, were it to occur, might facilitate cell‚Äìcell fusion activity, we mutated the wild-type SARS-CoV glycoprotein to construct a prototypic furin recognition site (RRSRR) at either position.</div>
                    </div></i></blockquote><div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/baa/50b/10f/baa50b10f2b1af00e1a1e8a10e23a17d.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
And the Japanese </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">inserted such a site</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (RRKR) into the SARS-CoV protein in 2008, though a bit downstream:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/493/013/0c7/4930130c77fc6bb955806ecd0dfb5c79.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the same 2008th year, their Dutch colleagues </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">also studied</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> these protease sites at SARS-CoV and compared them with the murine coronavirus MHV, which has this site (SRRAHR | SV), moreover, similar to the site of our CoV2 (SPRRAR | SV):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/83a/a21/8e9/83aa218e9ac0926d9bc514c47281184b.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In 2009, another American group also </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">decided to practice</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ‚Äúimproving‚Äù SARS-CoV and, without changing the American tradition of not trifling with arginines, they inserted RRSRR </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">:</font></font></b><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">To investigate the potential use of SARS-CoV S1-S2 and S2 'positions as sites for proteolytic cleavage, we first introduced furin cleavage recognition sites at these sites by making the following mutations 664-SLLRSTSQSI - SLL </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">RRSRR</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> SI-671 (S1-S2) and 792-LKPTKRSF-LKRTKRSF-799 (S2 ').</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">To examine the potential use of the SARS-CoV S1‚ÄìS2 and S2? positions as sites for proteolytic cleavage, we first introduced furin cleavage recognition sites at these locations by making the following mutations 664-SLLRSTSQSI ‚Äî SLL<b>RRSRR</b>SI-671 (S1‚ÄìS2) and 792-LKPTKRSF ‚Äî LKRTKRSF-799 (S2?).</div>
                    </div></i></blockquote><h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Beijing 2019</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But the most recent similar work that I saw was the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">work of October 2019 by</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> scientists from Beijing, where the new furin site RRKR was inserted not in some pseudovirus, but in the real chicken coronavirus, infectious bronchitis virus (IBV):</font></font><br>
<br>
<div style="text-align:center;"><img width="500" src="https://habrastorage.org/getpro/habr/post_images/c3d/b8a/566/c3db8a566877797ba5829cf23a039635.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, it is interesting to mention the authors that the addition of a furin site allows the mutant virus to infect nerve cells. Perhaps it is the CoV2 furin site that causes some patients with CoV2 to exhibit </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">neurological symptoms</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , including loss of smell:</font></font><br>
<blockquote><i>  S2'      QX     ,         ,     IBV (WT-IBV).       ,  <b> IBV    S2 ( -S2)     .</b>   IBV       .<br>
‚Ä¶<br>
 ,   ,     FP        CoV,  ,    ,   -,    ,   .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Mutation of the S2' site of QX genotype (QX-type) spike protein (S) in a recombinant virus background results in higher pathogenicity, pronounced neural symptoms and neurotropism when compared with conditions in wild-type IBV (WT-IBV) infected chickens. In this study, we present evidence suggesting that <b>recombinant IBV with a mutant S2' site (furin-S2' site) leads to higher mortality. </b>Infection with mutant IBV induces severe encephalitis and breaks the blood‚Äìbrain barrier.<br>
‚Ä¶<br>
In summary, our results demonstrate that the furin cleavage site upstream of the FP in S protein is an important site for CoV, modulating entry, cell‚Äìvirus fusion, adaptation to its host cell, cell tropism and pathogenicity, but not antigenicity.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Moreover, many coronaviruses have furin sites, of course, found in nature, and they are very diverse. Yes, and they can appear as a result of random mutations. This is exactly what happened in the case of MERS, which was </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">told to</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> us in 2015 by an </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">international team of authors</font></a><font style="vertical-align: inherit;"> , among whom were Shi Zhengli and Ralph Barik, two stars of synthetic coronavirusology. We will recall them more than once, but for now a few words about that article. Actually in it, the authors showed two key mutations that allowed MERS to jump from bats to humans. And one of these mutations led to the emergence of a furin site. True, this was not a box of new amino acids, but mutations of previously existing ones (marked in red below):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/ada/9e0/17c/ada9e017cc0e7c45c1bf0cfff3488f7f.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The authors did not just show, but attached these mutations back to the original bat-like spike protein, creating the same mutations in it (that is, the furin site), and showing that this gives it the ability to infect human cells:</font></font><br>
<blockquote><i>    ,   HKU4    , <b>    HKU4,          </b>.        , S746R  N762A,   HKU4.<br>
‚Ä¶<br>
<b>,   hPPC  hECP    HKU4   </b>      HKU4        .   ,   HKU4        S1/S2,          .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">To evaluate the potential genetic changes required for HKU4 to infect human cells, <b>we reengineered HKU4 spike, aiming to build its capacity to mediate viral entry into human cells.</b> To this end, we introduced two single mutations, S746R and N762A, into HKU4 spike. <b>The S746R mutation was expected to restore the hPPC motif in HKU4 spike</b>, whereas the N762A mutation likely disrupted the potential N-linked glycosylation site in the hECP motif in HKU4 spike.<br>
‚Ä¶<br>
<b>Therefore, the reengineered hPPC and hECP motifs enabled HKU4 spike to be activated by human endogenous proteases and thereby allowed HKU4 pseudoviruses to bypass the need for exogenous proteases to enter human cells. </b>These results reveal that HKU4 spike needs only two single mutations at the S1/S2 boundary to gain the full capacity to mediate viral entry into human cells.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">By the way, </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">how</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> they did this can frighten people far from modern biotechnology in itself - because the authors inserted this coronavirus spike-like protein into inactivated retrovirus (HIV):</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Briefly, pseudotyped MERS-CoV-spike retroviruses expressing the luciferase reporter gene were obtained by cotransfection of HEK293T cells with a plasmid carrying the HIV-1 gene, Env defective expressing luciferase (pNL4-3.luc.RE-) and a plasmid encoding MERS -CoV spike protein.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Briefly, MERS-CoV-spike-pseudotyped retroviruses expressing a luciferase reporter gene were prepared by cotransfecting HEK293T cells with a plasmid carrying Env-defective, luciferase-expressing HIV-1 genome (pNL4‚Äì3.luc.R-E-) and a plasmid encoding MERS-CoV spike protein.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Perhaps this is what prompted the Indian researchers to </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">search</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> for sequences similar in HIV and CoV2 in the genome (but their preprint was quickly criticized for the unsuccessful methodology and erroneous conclusions). </font><font style="vertical-align: inherit;">In fact, experts use such pseudoviruses regularly, and in general, one should not be indifferent to fear retroviruses as a class - their subspecies </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">lentiviruses have been</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> used for </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">gene therapy</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> for many years.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Where did RaTG13 come from</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In general, RaTG13 is a phenomenal strain. It is strange that all these years, Shi Zhengli's group was silent about him. After all, he does not at all resemble his other brethren, especially in the sequence of the spike-like protein, and this, I recall, is precisely the place that determines which types of cells (and which types) this virus can cling to. Here is a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">graph of the similarity</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> of the CoV2 genome compared to other bat coronaviruses (panel B):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/008/eaf/592/008eaf592574cba565f9d2f6ab5d4f79.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
RaTG13 is the red curve, and blue is the strains closest to RaTG13 (ZXC21 and ZC45). These strains </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">were isolated</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> from Chinese horseshoes ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Rhinolophus sinicus</font></font></i></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ) in </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Zhoushan</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in 2015 ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">ZXC21</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ) and 2017 ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">ZC45</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ). As can be seen from the graph, even they differ very strongly from RaTG13 (red curve) in the region of S protein. At the same time, the graph above does not so well convey the scale of the abyss between them as a direct comparison of sequences:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/ap/7l/yu/ap7lyu5fqeg_p8trsy4vqlmoix4.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As you can see, the spike-like proteins ZXC21 and ZC45 are not only generally 23-24 amino acid residues shorter than the RaTG13 protein, but they are shorter in the most important place - in RBM (see deletions in the red box marked with red dashes). </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
So where did the RaTG13 come from? As I said, in 2020, Shi Zhengli reported that she isolated it from horseshoe bats (only the species </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Rhinolophus </font></font></i><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">affinis</font></font></i></b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , not R. sinicus</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ) in July 2013 in Yunnan. True, until the end of January 2020, its existence was not publicly reported, but as the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">group Shi Zhengli</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> herself describes her significant discovery about its similarity to </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">CoV2</font></a><font style="vertical-align: inherit;"> :</font></font><br>
<blockquote><i>  ,    - - (RdRp)     (BatCoV RaTG13),      Rhinolophus affinis   ,     2019-CoV2.         (  GISAID EPI_ISL_402131).  Simplot ,  2019-CoV2    RaTG13 (Fig. 1c),     96,2%.<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">We then found that a short region of RNA-dependent RNA polymerase (RdRp) from a bat coronavirus (BatCoV RaTG13) ‚Äî which was previously detected in Rhinolophus affinis from Yunnan province ‚Äî showed high sequence identity to 2019-CoV2. We carried out full-length sequencing on this RNA sample (GISAID accession number EPI_ISL_402131). Simplot analysis showed that 2019-CoV2 was highly similar throughout the genome to RaTG13 (Fig. 1c), with an overall genome sequence identity of 96.2%.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Not thick: well, they had this strain </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">"previously discovered"</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , and it was. </font><font style="vertical-align: inherit;">Lying on a shelf. </font><font style="vertical-align: inherit;">Until 2020, only the part of the genome responsible for RdRp was sequenced in it. </font><font style="vertical-align: inherit;">Where did he come from on the shelf? </font><font style="vertical-align: inherit;">In Yunnan in 2013 allocated. </font><font style="vertical-align: inherit;">Where exactly? </font><font style="vertical-align: inherit;">They did not say. </font><font style="vertical-align: inherit;">In </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">GenBank</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , too, did not have this information. </font><font style="vertical-align: inherit;">But, fortunately, there was a GISAID in the genome database: it is reported that in the city of Puer (yes, in the homeland of Basta's beloved tea), a male was isolated from fecal swab:</font></font><br>
<br>
<div style="text-align:center;"><img width="450" src="https://habrastorage.org/getpro/habr/post_images/23c/bb3/b86/23cbb3b86305a9572ecccfcbc8a22305.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
This intrigued me a little, because in my wanderings around Pabmed, I had already stumbled upon an </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">expedition to Puer in the summer of 2013</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> :</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Bats were caught in various places in five districts of the four prefectures of Yunnan, China, from May to July 2013.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Bats were captured from various locations in five counties of four prefectures of Yunnan Province, China, from May to July 2013.</div>
                    </div></i></blockquote><div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/c17/74b/6bf/c1774b6bf95d0517b96c6f9ddcc7024e.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Researchers did not find anything particularly interesting for us on that expedition, but perhaps it was then that Shi Zhengli (or someone from her group?) Identified the same RaTG13 sample? Which they sequenced only partially, but for some reason decided not to publish, although it was very different from everything previously known. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Shi Zhengli herself could well personally participate in that expedition, because she spoke very warmly about such expeditions - for example, in her </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">TED-like performance</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in 2018, where she showed her photos from such expeditions:</font></font><br>
<br>
<iframe width="560" height="315" src="https://www.youtube.com/embed/mvbyRdRRb7o" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It was a series of such expeditions that </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">brought her worldwide fame</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> and the nickname ‚ÄúBatumen‚Äù: in a 2013 </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">article in Nature</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , the Shi Zhengli group triumphantly announced that in the caves of Yunnan she had discovered carrier bats of strains RsSHC014 and Rs3367 that coincided with the first SARS -CoV by 85% and 96%, respectively. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, it is a funny coincidence that at about the same time, in the same Yunnan, the Shi Zhengli group also turned out to find the RaTG13 strain, which turned out to be the closest to CoV2, and their genomes also coincide 96%.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Wuhan-1</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Returning to that triumphant article from 2013, Shi Zhengli's group also said that by cultivating the isolated samples in </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Vero monkey cell</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> culture </font><font style="vertical-align: inherit;">, they managed to isolate a live virus that was almost identical to the Rs3367 strain (closest to SARS-CoV) . </font><font style="vertical-align: inherit;">The authors dubbed their offspring WIV1 (where WIV stands for Wuhan Institute of Virology):</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Most importantly, we report the first isolation of live SL-CoV virus (bat SL-CoV-WIV1) from samples isolated from bat feces in Vero E6 cells, which has a typical morphology of coronavirus, genome identity of 99.9% s Rs3367 and uses ACE2 people, civet and horseshoe bones to enter the cell. </font><font style="vertical-align: inherit;">Preliminary in vitro testing shows that WIV1 also has wide species tropism.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Most importantly, we report the first recorded isolation of a live SL-CoV (bat SL-CoV-WIV1) from bat faecal samples in Vero E6 cells, which has typical coronavirus morphology, 99.9% sequence identity to Rs3367 and uses ACE2 from humans, civets and Chinese horseshoe bats for cell entry. Preliminary in vitro testing indicates that WIV1 also has a broad species tropism.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Let's compare RaTG13 with strains Rs3367 and RsSHC014:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/bo/ad/gm/boadgmonhp0uum1xeg7_en8ihri.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As you can see, the spike-like protein of these strains is not only shorter than 13 amino acids RaTG13-shn, but it also differs greatly in its first quarter. </font><font style="vertical-align: inherit;">By the way, it is curious that the spike proteins in Rs3367 (aka WIV1) and RsSCH014 are almost identical, and differ only in the RBD region (right sequence below). </font><font style="vertical-align: inherit;">Almost like CoV2 and RaTG13 (not counting the furin insert):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/ju/uw/fq/juuwfqel6vk6tvqilou7ezrze4i.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Could any researcher, having received coronavirus samples from customs confiscated pangolins in March 2019, want to check how RBM pangolinium is tropic to the human receptor? </font><font style="vertical-align: inherit;">And what if with a polybasic furin site in the most interesting place? </font><font style="vertical-align: inherit;">This will be a bomb!</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Theoretically, of course, he could. </font><font style="vertical-align: inherit;">There is nothing technically difficult for virologists to carry out such experiments. </font><font style="vertical-align: inherit;">A reasonable question: why would they use RaTG13 as a basis, and not already tested WIV1? </font><font style="vertical-align: inherit;">But it is possible that they also tested the chimera with WIV1. </font><font style="vertical-align: inherit;">And in parallel, they decided to simulate the variant of recombination of the pangolin virus with the bat closer to it - after all, RaTG13 is nevertheless closer to the pangolin strains than WIV1: its spike-like protein is closer to them both phylogenetically and structurally - it even matches them in length, while the proteins WIV1 / Rs3367 and RsSHC014 13 amino acids shorter than pangolin. </font><font style="vertical-align: inherit;">And the QTQTNS mutation common to RaTG13 and pangolin-19 (MP789) in the region of the protease site cannot leave indifferent the specialist.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Other Yunnan strains</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, other researchers in 2011 </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">also found</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> samples of coronaviruses from the Yunnan </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Rhinolophus </font></font></i><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">affinis</font></font></i></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . The strain LYRa11 seemed to me the most interesting:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/0b8/cbe/28f/0b8cbe28f6e6e41435088ac92d014a7c.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But it is also very far from RaTG13, and much closer to Rs3367 (this is the strain that 96% coincides with the first SARS-CoV):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/x5/b6/06/x5b606tbipv1rn-pqectcw2vmzk.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But RaTG13, isolated from the same bats of </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Rhinolophus </font></font></i><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">affinis</font></font></i></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> as LYRa11, looks the least like it (left blast). </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
And finally, another Yunnan strain (ingenuously named Yunnan2011), </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">isolated in 2011</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> from another subspecies of the horseshoe </font><font style="vertical-align: inherit;">race </font><font style="vertical-align: inherit;">, </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Rhinolophus </font></font></i><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">pusillus</font></font></i></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , is similar to RaTG13 even less than LYRa11:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/-g/qv/fl/-gqvflspt2n8vm8msfo7as09lfk.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Yes, and between themselves, Yunnan2011 and LYRa11 (the right blast above) are not particularly similar, apart from the highly conserved region S2. </font><font style="vertical-align: inherit;">By the way, what kind of mess do they have with the names of the strains? </font><font style="vertical-align: inherit;">First they prescribe it all year, sometimes partially, or even not at all (Rs3367). </font><font style="vertical-align: inherit;">First comes the type of carrier ( </font></font><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Ra</font></font></i></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> TG13), then later (LY </font></font><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Ra</font></font></i></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 11). </font><font style="vertical-align: inherit;">Still wondering what TG, LY or SHC mean? </font><font style="vertical-align: inherit;">Initials decrypting the genome? </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Alright, let's move on from viral archeology to viral engineering, namely, transplantation of key areas of the spike protein and other gain-of-function (GOF) experiments.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">1999: First Chimeric Coronavirus</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
If you think that all this GOF-engine with the analysis of what exactly allows coronaviruses to jump from one species to another began in response to the epidemic of the first SARS in 2002, you are mistaken. Virologists experimented with chimeric coronaviruses long before that. Here, for example, is a </font><font style="vertical-align: inherit;">sample </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> from 1999 from the Dutch group of Peter Rottier from Utrecht with the saying ‚ÄúRetargeting coronavirus by replacing the ectodomain in the spike protein: crossing the species barrier‚Äù:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Using directed RNA recombination, we constructed a mutant of the coronavirus mouse hepatitis virus (MHV), in which the spine-shaped ectodomain was replaced by the highly divergent ectodomain of the feline infectious peritonitis virus protein. </font><font style="vertical-align: inherit;">The resulting chimeric virus, designated fMHV, acquired the ability to infect feline cells and at the same time lost the ability to infect mouse cells in culture.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Using targeted RNA recombination, we constructed a mutant of the coronavirus mouse hepatitis virus (MHV) in which the ectodomain of the spike glycoprotein (S) was replaced with the highly divergent ectodomain of the S protein of feline infectious peritonitis virus. The resulting chimeric virus, designated fMHV, acquired the ability to infect feline cells and simultaneously lost the ability to infect murine cells in tissue culture.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">By the way, Shi Zhengli seems to have had an internship under Peter Rottier in Utrecht. </font><font style="vertical-align: inherit;">At least in 2005, she was co-author of a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">joint article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> where Utrecht was listed as her affiliate (but the current address was already indicated at the Shanghai Institute). </font><font style="vertical-align: inherit;">By the way, the article itself is also very curious - in it the authors investigated what exactly allows viruses to expand their species tropism:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Only a small number of mutations in its spike-like protein allows the mouse coronavirus to switch from mouse-limited tropism to an extended host range by in vitro passivation. </font><font style="vertical-align: inherit;">One such virus that we studied acquired two putative heparan sulfate </font><b><font style="vertical-align: inherit;">binding</font></b><font style="vertical-align: inherit;"> sites, </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">while preserving the furin site elsewhere.</font></font></b><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Only a relatively few mutations in its spike protein allow the murine coronavirus to switch from a murine-restricted tropism to an extended host range by being passaged in vitro. One such virus that we studied had acquired two putative heparan sulfate-binding sites <b>while preserving another site in the furin-cleavage motif</b>.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Firstly, it is interesting that the furin site in that virus (SRRAHR | SV) is similar to the site in CoV2 (SPRRAR | SV), although it cuts more efficiently in CoV2 due to dual arginines (this makes it a </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">polybasic</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> site, i.e. he has </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">several </font></font></b> <b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">basic</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> amino acids in a row </font><font style="vertical-align: inherit;">in the RxxR sequence </font><font style="vertical-align: inherit;">):</font></font><br>
<br>
<div style="text-align:center;"><img width="450" src="https://habrastorage.org/getpro/habr/post_images/03f/e6b/fc2/03fe6bfc2fad1d03fd3a0b16bc538ab9.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But the article is especially curious that the mutations that allowed the virus to ‚Äúbroaden its horizons‚Äù did not even occur in laboratory animals, but in vitro ( </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">by being passaged in vitro</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ). And, it seems, they happened pretty quickly:</font></font><br>
<blockquote><i>MHV/pi23 ‚Äî ,   23  600 ,   MHV/BHK,    HS-    S1    ,    MHV/BHK,      ,         HS-     . MHV/pi23     ,    ,  MHV/BHK.     HS- , , ,      S,    HR1     (Fig. 1),        . <b>          S,       .</b><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">MHV/pi23, a virus obtained after 23 of the 600 passages that resulted in MHV/BHK, also contains a putative HS-binding site in the S1 domain at the same position as in MHV/BHK, albeit as a smaller insertion, while it lacks the putative HS-binding site immediately upstream of the fusion peptide. MHV/pi23 does infect nonmurine cells to some extent but much less efficiently than MHV/BHK. In addition to the multiple HS-binding sites, however, mutations found in other parts of the S protein, such as the HR1 domain and the putative fusion peptide (Fig. 1), might also contribute to the efficient entry into nonmurine cells. We are currently in the process of determining the S protein mutations that are required for the extended host range phenotype.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Looking ahead, I‚Äôll mention that there were other groups that </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">tried in vitro mutations to</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> increase the virulence of the coronavirus, for example, MERS:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">To better understand the adaptability of MERS-CoV species, we used the suboptimal DPP4 variant to study viral adaptation. </font><font style="vertical-align: inherit;">Passaging of the virus on cells expressing this DPP4 variant resulted in the accumulation of mutations in the viral spike protein, which increased replication.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">To better understand the species adaptability of MERS-CoV, we identified a suboptimal species-derived variant of DPP4 to study viral adaption. Passaging virus on cells expressing this DPP4 variant led to accumulation of mutations in the viral spike which increased replication.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Moreover, their mutations occurred already after several </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">passages</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (rounds of reproduction of cell cultures):</font></font><br>
<br>
<div style="text-align:center;"><img width="450" src="https://habrastorage.org/getpro/habr/post_images/0b8/29e/754/0b829e754f3362ba124def01048d0bba.png"></div><blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">(F) Scheme of the appearance of single and double mutations in the MERS-CoV spike protein in different passages. </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
(G) Location of mutations in the MERS-CoV spike protein.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">(F) Schematic of single and double mutation emergence in MERS-CoV spike over different passages. <br>
(G) Location of mutations within MERS-CoV spike.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">But all this will be much later. </font><font style="vertical-align: inherit;">In the meantime, let's get back to 2002 - BEFORE the outbreak of the first SARS-CoV.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">How Barik lit the way for everyone</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Ralph Barik is a legend in coronavirusology. </font><font style="vertical-align: inherit;">A true pioneer in synthetic viral genome manipulation techniques. </font><font style="vertical-align: inherit;">Back in 2002, he published a breakthrough work that opened a new milestone both in the study of various mechanisms of natural viruses and in gain-of-function (GOF) research. </font><font style="vertical-align: inherit;">In </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">their work</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , Barik's group synthetically recreated a clone of the natural mouse coronavirus:</font></font><br>
<blockquote><i>             II  59 (MHV-A59).     ,     MHV  31,5   .              ,         MHV-A59  ~ 31,5 ...     ,       ,         ,         ‚Ä¶         ,    ,                    .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">A novel method was developed to assemble a full-length infectious cDNA of the group II coronavirus mouse hepatitis virus strain A59 (MHV-A59). Seven contiguous cDNA clones that spanned the 31.5-kb MHV genome were isolated. The ends of the cDNAs were engineered with unique junctions and assembled with only the adjacent cDNA subclones, resulting in an intact MHV-A59 cDNA construct of ?31.5 kb in length. The interconnecting restriction site junctions that are located at the ends of each cDNA are systematically removed during the assembly of the complete full-length cDNA product, allowing reassembly without the introduction of nucleotide changes‚Ä¶ The method has the potential to be used to construct viral, microbial, or eukaryotic genomes approaching several million base pairs in length and used to insert restriction sites at any given nucleotide in a microbial genome.</div>
                    </div></i></blockquote><div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/fa7/bdb/96a/fa7bdb96a4fe78a56d185fe48c5c809c.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
That is, the authors, in fact, translated the RNA virus into the language of DNA (using reverse transcriptase), so that then it would be convenient for them to manipulate its genome using existing genetic engineering tools. </font><font style="vertical-align: inherit;">Having created 7 such cDNA provirus segments, the authors then stitched them together ‚Äúseamlessly‚Äù (without leaving any traces), after which they transferred their construct back to RNA, from which viral particles were then formed in other cells.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">SARS-2003</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Just a few weeks after the publication of that work by Barik's group, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">the</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> SARS-CoV </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">epidemic</font></a><font style="vertical-align: inherit;"> struck </font><font style="vertical-align: inherit;">, and Ralph Barik wasted no time. </font><font style="vertical-align: inherit;">Already in the summer of 2003, his group sent to print the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">work</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on creating a synthetic clone SARS-CoV:</font></font><br>
<blockquote><i>    ,    ,      SARS-CoV Urbani      SARS (  SARS-CoV),     ,   .      ,    ,       ‚Ä¶    SARS-CoV      ,                 .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Using a panel of contiguous cDNAs that span the entire genome, we have assembled a full-length cDNA of the SARS-CoV Urbani strain, and have rescued molecularly cloned SARS viruses (infectious clone SARS-CoV) that contained the expected marker mutations inserted into the component clones. Recombinant viruses replicated as efficiently as WT virus and both were inhibited by treatment with the cysteine proteinase inhibitor‚Ä¶ Availability of a SARS-CoV full-length cDNA provides a template for manipulation of the viral genome, allowing for the rapid and rational development and testing of candidate vaccines and therapeutics against this important human pathogen.</div>
                    </div></i></blockquote><div style="text-align:center;"><img width="450" src="https://habrastorage.org/getpro/habr/post_images/555/fe0/c96/555fe0c96cf0c77a57574d3027ba2c31.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The speed of the Barik group allows us to understand how quickly a qualified team of virologists can create a synthetic clone from a natural virus, and therefore make genetic modifications to it. </font><font style="vertical-align: inherit;">And it was in 2003. </font><font style="vertical-align: inherit;">Today all the same qualified laboratory can repeat in a matter of weeks.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">SARS-2006</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Barik was the first, but far from the last. </font><font style="vertical-align: inherit;">Genetic engineering has developed by leaps and bounds, creating more and more new tools. </font><font style="vertical-align: inherit;">Other groups practiced alternative synthetic virology technologies. </font><font style="vertical-align: inherit;">For example, in 2006, Spanish researchers repeated the achievements of Barik, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">also creating a synthetic SARS clone</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , but using another technology ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">artificial bacterial chromosome</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ):</font></font><br>
<blockquote><i>             Urbani  SARS-CoV      (BAC).                    .   ,       Escherichia coli.<br>
‚Ä¶<br>
    SARS-CoV        E.coli DH10B    200 ,         (  ).         .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">The engineering of a full-length infectious cDNA clone and a functional replicon of the severe acute respiratory syndrome coronavirus (SARS-CoV) Urbani strain as bacterial artificial chromosomes (BACs) is described in this study. In this system, the viral RNA was expressed in the cell nucleus under the control of the cytomegalovirus promoter and further amplified in the cytoplasm by the viral replicase. Both the infectious clone and the replicon were fully stable in Escherichia coli. <br>
‚Ä¶<br>
The assembled SARS-CoV infectious cDNA clone was fully stable during its propagation in E. coli DH10B cells for more than 200 generations, considerably facilitating the genetic manipulation of the viral genome (data not shown). The detailed cloning strategy, plasmid maps, and sequences are available upon request.</div>
                    </div></i></blockquote><div style="text-align:center;"><img width="500" src="https://habrastorage.org/getpro/habr/post_images/8a4/27f/e2c/8a427fe2c86807b79be6a32efde8f56e.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
True, they didn‚Äôt do it as elegantly as Barik, since in the final assembly of the synthetic virus they still had added restriction enzyme sites, while Barik learned to combine the fragments ‚Äúseamlessly‚Äù. </font><font style="vertical-align: inherit;">But these are trifles, the Spaniards approach is also quite working - in 2013, with their help, they </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">created a synthetic clone of MERS-a</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , and in 2015, their technique was included in the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">coronavirus textbook-guide</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (chapter 13):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/529/757/b5b/529757b5b59118788ed05ebcee5c4e8a.png"></div><br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Wuhan 2007</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But back in 2007. </font><font style="vertical-align: inherit;">Then, the Shi Zhengli group joined the synthetic virology race with a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">study of the spike-like protein of</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> human and bat coronaviruses, trying to determine what exactly is responsible for the ability to skip from species to species:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">A number of chimeric spike proteins have been constructed by inserting various SARS-CoV spike protein sequences into the backbone of the SL-CoV protein.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">A series of S chimeras was constructed by inserting different sequences of the SARS-CoV S into the SL-CoV S backbone.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">That is, the authors inserted different segments from the human SARS-CoV protein into the protein of the bat virus. </font><font style="vertical-align: inherit;">Here is their conclusion:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">From these results, it was found that the region from 310 to 518 in the spike protein BJ01 was necessary and sufficient to obtain the spike protein Rp3 ability to bind to human ACE2.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">From these results, it was deduced that the region from aa 310 to 518 of BJ01-S was necessary and sufficient to convert Rp3-S into a huACE2-binding molecule.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">At the same time, they tried to replace shorter fragments, including only RBM:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">To insert RBM from SARS-CoV into the SL-CoV spike protein, the coding region of 424 to 494 amino acids in the spike protein BJ01 was used to replace the corresponding regions in the Rp3 protein, resulting in a chimeric spike protein gene (CS), designated as CS424 ‚Äì494.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">For introduction of the RBM of SARS-CoV S into the SL-CoV S, the coding region from aa 424 to 494 of BJ01-S was used to replace the corresponding regions of Rp3-S, resulting in a chimeric S (CS) gene designated CS424‚Äì494.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Given that it was written in 2007, I think today it will not be difficult for even a novice virologist to replace the RBM of one virus with another.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Chimera 2015</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the light of the above experiments, it is not very clear what exactly caused the sensation that the probably most sensational gain-of-function publication produced. </font><font style="vertical-align: inherit;">This, of course, is about the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">joint work of</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> Shi Zhengli and Ralph Barik, in which they created a synthetic chimeric virus:</font></font><br>
<blockquote><i>    SARS-CoV,      ,       SHC014    SARS-CoV.  ,    2b,    SHC014    ,        SARS (ACE2),            in vitro,    SARS-CoV.  ,  in vivo          .             ;     ,            CoV    .            SHC014      in vitro,   in vivo.<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Using the SARS-CoV reverse genetics system, we generated and characterized a chimeric virus expressing the spike of bat coronavirus SHC014 in a mouse-adapted SARS-CoV backbone. The results indicate that group 2b viruses encoding the SHC014 spike in a wild-type backbone can efficiently use multiple orthologs of the SARS receptor human angiotensin converting enzyme II (ACE2), replicate efficiently in primary human airway cells and achieve in vitro titers equivalent to epidemic strains of SARS-CoV. Additionally, in vivo experiments demonstrate replication of the chimeric virus in mouse lung with notable pathogenesis. Evaluation of available SARS-based immune-therapeutic and prophylactic modalities revealed poor efficacy; both monoclonal antibody and vaccine approaches failed to neutralize and protect from infection with CoVs using the novel spike protein. On the basis of these findings, we synthetically re-derived an infectious full-length SHC014 recombinant virus and demonstrate robust viral replication both in vitro and in vivo.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">That is, in fact, the researchers were already on the beaten path: they took the spike-like protein from RsSHC014, which Shi Zhengli isolated from Yunnan bats in 2011, and inserted it into SARS-CoV, specially adapted for crawling mice, for subsequent </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in vivo</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> experiments on these very mice. </font><font style="vertical-align: inherit;">Well, on human cells, a new design was tested. </font><font style="vertical-align: inherit;">And at the same time we practiced creating a recombinant clone of the same RsSHC014 - well, why not? </font><font style="vertical-align: inherit;">After all, firstly, it is beautiful:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/7f5/d39/f95/7f5d39f95fa1168fa0fbe4491df69456.png"></div><blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">(a) Scheme of the molecular clone SHC014-CoV, which was synthesized in the form of six contiguous cDNA segments (designated as SHC014A, SHC014B, SHC014C, SHC014D, SHC014E and SHC014F) flanked by unique BglI restriction enzyme sites that provided directional assembly for DNA 1a, 1b, spike, 3, envelope, matrix, 6‚Äì8 and nucleocapsid). </font><font style="vertical-align: inherit;">Underlined nucleotides are protruding sequences formed after restriction enzyme digestion.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">(a) Schematic of the SHC014-CoV molecular clone, which was synthesized as six contiguous cDNAs (designated SHC014A, SHC014B, SHC014C, SHC014D, SHC014E and SHC014F) flanked by unique BglI sites that allowed for directed assembly of the full-length cDNA expressing open reading frames (for 1a, 1b, spike, 3, envelope, matrix, 6‚Äì8 and nucleocapsid). Underlined nucleotides represent the overhang sequences formed after restriction enzyme cleavage.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">And secondly, the researchers realized that it‚Äôs not only by the tenon nature of the spike-like protein to the receptor that the virus‚Äôs potential for transition from one animal species to another is determined because the SHC014-MA15 chimera was more virulent than SHC014 itself, even in human cells:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">It is noteworthy that the differential tropism in the lungs compared to that of SARS-MA15 and the weakening of full-length SHC014-CoV in cultures [human epithelial airways] relative to SARS-CoV Urbani suggest that in addition to binding to ACE2, other factors - including the </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">processivity of the spike protein</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , bioavailability of the receptor or antagonism of the host's immune responses - may contribute to virulence.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Notably, differential tropism in the lung as compared to that with SARS-MA15 and attenuation of full-length SHC014-CoV in [human epithelial airway cell] cultures relative to SARS-CoV Urbani suggest that factors beyond ACE2 binding ‚Äî <b>including spike processivity</b>, receptor bio-availability or antagonism of the host immune responses ‚Äî may contribute to emergence.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">I especially want to highlight the processivity of the spike protein in the quote, because this is not the first time that researchers have written that the ability of the spike protein to cleave proteases (including furin) has a significant effect on virulence. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In conclusion, the theme is a joint photo of Ralph Barik and Shi Zhengli. </font><font style="vertical-align: inherit;">Photo taken in Wuhan, in October 2018:</font></font><br>
<br>
<div style="text-align:center;"><img width="800" src="https://habrastorage.org/webt/9o/uj/xu/9oujxuxgzxlia8o8t6fdjcsygu4.png"></div><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Mouse SARS-2007</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Here I can‚Äôt help but mention what kind of ‚Äúmouse virus MA15‚Äù it was in a previous work. </font><font style="vertical-align: inherit;">This is not at all some kind of natural mouse coronavirus, as one might suggest. </font><font style="vertical-align: inherit;">And this is a laboratory-modified human SARS-CoV, which back in 2007, the same Barik group - apparently competing with the Shi Zhengli group (remember their article from 2007) - </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">turned into a real murderer</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">To do this, they first iteratively ‚Äúimproved‚Äù on mice, and when after several iterations it became maximally ‚Äúeffective‚Äù, they reproduced the mutations that arose in mice in the synthetic clone of a new virus, and once again checked that it really has increased infectivity:</font></font><br>
<blockquote><i>  SARS-CoV ( Urbani)         BALB/c.      (MA15),        .          ,       ,  ,      .            ,               .   ,  ,  15,       ,        .  MA15    ,      ; <b>    SARS-CoV         (rMA15)</b>,      MA15.   15    ,     SARS  .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">We adapted the SARS-CoV (Urbani strain) by serial passage in the respiratory tract of young BALB/c mice. Fifteen passages resulted in a virus (MA15) that is lethal for mice following intranasal inoculation. Lethality is preceded by rapid and high titer viral replication in lungs, viremia, and dissemination of virus to extrapulmonary sites accompanied by lymphopenia, neutrophilia, and pathological changes in the lungs. Abundant viral antigen is extensively distributed in bronchial epithelial cells and alveolar pneumocytes, and necrotic cellular debris is present in airways and alveoli, with only mild and focal pneumonitis. These observations suggest that mice infected with MA15 die from an overwhelming viral infection with extensive, virally mediated destruction of pneumocytes and ciliated epithelial cells. The MA15 virus has six coding mutations associated with adaptation and increased virulence; <b>when introduced into a recombinant SARS-CoV, these mutations result in a highly virulent and lethal virus (rMA15)</b>, duplicating the phenotype of the biologically derived MA15 virus. Intranasal inoculation with MA15 reproduces many aspects of disease seen in severe human cases of SARS.</div>
                    </div></i></blockquote><br>
<h3>-2008</h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Speaking of the rivalry of Barik and Shi Zhengli. In parallel with the transplantation of RBD from human SARS-CoV to murine, his group created the same chimeras with bat strains. In 2008, Barik's group </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">took the</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> Bat-SCoV strain and replaced it with RBD in a spike protein with RBD from human SARS. That is, in fact, she repeated the work of Shi Zhengli from 2007, not only limiting herself to pseudo-viruses, but creating a real chimeric coronavirus - not just a murine one like in the work of 2015, but the most human one. The authors were clearly proud of their achievement:</font></font><br>
<blockquote><i>    ,           29,7  ‚Äî    (Bat-SCoV),      (SARS),    SARS-CoV.<br>
‚Ä¶<br>
 ,   RBDs Bat-SCoV  SARS-CoV ,   RBD Bat-SCoV ( 323‚Äì505)  RBD SARS-CoV ( 319‚Äì518) (27, 28) (GenBank   FJ211860),   ,        in vivo (Fig. 1B).<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Here, we report the design, synthesis, and recovery of the largest synthetic replicating life form, a 29.7-kb bat severe acute respiratory syndrome (SARS)-like coronavirus (Bat-SCoV), a likely progenitor to the SARS-CoV epidemic.<br>
‚Ä¶<br>
To test whether the RBDs of Bat-SCoV and SARS-CoV were interchangeable, we replaced the Bat-SCoV RBD (amino acid 323‚Äì505) with the SARS-CoV RBD (amino acid 319‚Äì518) (27, 28) (GenBank accession no. FJ211860), simulating a theoretical recombination event that might occur during mixed infection in vivo (Fig. 1B).</div>
                    </div></i></blockquote><div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/6bf/ee4/c3a/6bfee4c3a4a94bf60a525a1b22620456.png"></div><blockquote><i>(B)  ,     SARS-CoV  Bat-SCoV.        . Bat-SRBD      Bat-SCoV,   ,  RBD Bat-SCoV ( Bat-SCoV 323‚Äì505)   RBD SARS-CoV ( 319‚Äì518) (  GenBank FJ211860 ). Bat-SRBD-MA   RBD    MA15  SARS-CoV Y436H. Bat-SRBM   13  SARS-CoV,     ACE2,     RBD   323I-429T Bat-SCoV   426R-518D SARS-CoV. Bat-Hinge    Bat-SRBM,   Bat-SCoV 392L-397E    SARS-CoV 388V-393D. Bat-F    1‚Äì24057 SARS-CoV (  855   )   3'-  Bat-SCoV.               1-  (P1). ND ,     .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">(<i>B</i>) Schematic representation showing organization of the SARS-CoV and Bat-SCoV Spike proteins. The engineered Spike proteins are pictured below with the virus name to the left. Bat-SRBD includes all of the Bat-SCoV Spike sequence except that the Bat-SCoV RBD (Bat-SCoV amino acid 323‚Äì505) is replaced with the SARS-CoV RBD (amino acid 319‚Äì518) (GenBank accession no. <a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow">FJ211860</a>). Bat-SRBD-MA includes the MA15 Spike RBD change at SARS-CoV aa Y436H. Bat-SRBM includes the minimal 13 SARS-CoV residues critical for ACE2 contact, resulting in a chimeric RBD of Bat-SCoV amino acid 323I-429T and SARS-CoV amino acid 426R-518D. Bat-Hinge is Bat-SRBM sequence, with Bat-SCoV amino acid 392L-397E replaced with SARS-CoV amino acid 388V-393D. Bat-F includes nt 1‚Äì24057 of SARS-CoV (to Spike amino acid 855), with the remaining 3? sequence from Bat-SCoV. To the right of the schematic representations, observation of transcript activity and approximate stock titers at passage 1 (P1) are indicated. ND indicates no infectious virus detected by plaque assay.</div>
                    </div></i></blockquote><h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Barik 2016</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the work of Barik in general there are many remakes. </font><font style="vertical-align: inherit;">For example, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in 2016, the year</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> he almost repeated the very same work with Shi Chzhenli by 2015 to create a chimeric virus, but this time he put in myshinoadaptirovanny SARS piece spinous protein not from RsSCH014, and from another found Shi Chzhenli </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in Yunnan</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> it close relative - Rs3367. </font><font style="vertical-align: inherit;">Or, to be exact, from strain WIV1, a laboratory clone of Rs3367 grown in cell culture at the Wuhan Institute of Virology in 2013. </font><font style="vertical-align: inherit;">Here is exactly what Barik's group did in 2016:</font></font><br>
<blockquote><i>   SARS-CoV    (7),        WIV1-CoV,    ,      ,      ,      (. S1A).           WIV1-CoV,      SARS    WIV1      (WIV1-MA15, . S1B). ‚Ä¶        Vero   SARS-CoV Urbani, WIV1-MA15  WIV1-CoV.<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Using the SARS-CoV infectious clone as a template (7), we designed and synthesized a full-length infectious clone of WIV1-CoV consisting of six plasmids that could be enzymatically cut, ligated together, and electroporated into cells to rescue replication competent progeny virions (Fig. S1A). In addition to the full-length clone, we also produced WIV1-CoV chimeric virus that replaced the SARS spike with the WIV1 spike within the mouse-adapted backbone (WIV1-MA15, Fig. S1B). ‚Ä¶ To confirm growth kinetics and replication, Vero cells were infected with SARS-CoV Urbani, WIV1-MA15, and WIV1-CoV.</div>
                    </div></i></blockquote><div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/294/b21/ee9/294b21ee9dbb3d6f387d458c8ce1146c.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
That is, by and large, in 2016, Barik cloned his article from 2015. </font><font style="vertical-align: inherit;">Moreover, its meaning is not very clear to me: after all, WIV1 / Rs3367, if you remember, already 96% coincided with SARS-CoV (for this Shi Zhengli gained fame). </font><font style="vertical-align: inherit;">Therefore, why it is not very clear to me to insert a spike-like protein from its closest relative back into SARS-CoV. </font><font style="vertical-align: inherit;">Perhaps just out of a love of art. </font><font style="vertical-align: inherit;">In this light, the title of his article acquires a certain duality: </font></font><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">‚ÄúSARS-like WIV1-CoV is ready for the transition to humans‚Äù</font></font></i></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">For some reason, this frame immediately comes to mind:</font></font><br>
<br>
<div style="text-align:center;"><img width="700" src="https://habrastorage.org/getpro/habr/post_images/2df/91c/c29/2df91cc299208501cd736b4c3652433e.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
I still don‚Äôt understand how in 2015 Barik managed to </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">get a patent</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> for the creation of ‚Äúchimeric coronavirus spike-like proteins‚Äù, taking into account the fact that he and Shi Zhengli published all this long before 2015.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Barik 1990</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Okay, the final touch to the portrait of Ralph Barik. </font><font style="vertical-align: inherit;">He was not just an old-timer in this area, but was engaged in the design of recombinant coronaviruses even before any sequencers and other modern tools of genetic engineering. </font><font style="vertical-align: inherit;">Here is </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">his article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on the creation of ‚Äútemperature mutants‚Äù from mouse coronavirus from the year 1990:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Throughout this study, the mouse hepatitis virus strain A59 (MHV-A59) was used. </font><font style="vertical-align: inherit;">The virus was propagated and cloned three times in a continuous mouse astrocytoma cell line (DBT). </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
... </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Various combinations of temperature-sensitive mutants were mixed and inoculated into cells with a multiplicity of infection equal to 10.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">The A59 strain of mouse hepatitis virus (MHV-A59) was used throughout the course of this study. Virus was propagated and cloned three times in the continuous murine astrocytoma cell line (DBT). <br>
‚Ä¶<br>
Various combinations of ts mutants were mixed and inoculated onto cells at a multiplicity of infection of 10 each.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">So Barik has been creating various viral mutants for over 30 years.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Barik 2019</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
And, by the way, even now, the pace does not reduce. At the end of October 2019, his group submitted for publication another </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">article on the important role of the furin site</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in the spike protein to overcome the ‚Äúbarrier to zoonotic infection‚Äù by coronaviruses:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Together, these results demonstrate that protease cleavage is also a major barrier to Vero cell infection with HKU5-CoV. </font><font style="vertical-align: inherit;">Looking further, we compared the predicted cleavage at the S1 / S2, S2 'border and the endosomal cysteine ‚Äã‚Äãprotease site in the MERS, PDF2180, and HKU5 spike-like proteins (Fig. 6D) (26). </font><font style="vertical-align: inherit;">At the S1 / S2 site, MERS, Uganda, and HKU5 maintain the RXXR cleavage site, although various internal amino acids may affect its effectiveness. </font><font style="vertical-align: inherit;">For the S2 'sequence, MERS and HKU5 also retain the RXXR motif; </font><font style="vertical-align: inherit;">however, the first arginine (SNAR) is absent in the spikes of the Uganda strain, which potentially affects its cleavage.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Together, these results demonstrate that protease cleavage is also the primary barrier to infection of Vero cells with HKU5-CoV. Examining further, we compared the predicted cleavage at S1/S2 border, S2‚Äô, and the endosomal cysteine protease site across MERS, PDF2180, and HKU5 spikes (Fig. 6D) (26). For the S1/S2 site, MERS, Uganda, and HKU5 maintain the RXXR cleavage motif, although the different interior amino acids may alter efficiency. For the S2‚Äô sequence, MERS and HKU5 also retain the RXXR motif; however, the Uganda spike lacks the first arginine (SNAR), potentially impacting cleavage.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Mindful of the competitive spirit between the groups of Barik and Shi Zhengli, I wonder what is the likelihood that in Wuhan at the end of 2019 someone also did similar research?</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Gain-of-Function: ‚ÄúDeny cannot continue‚Äù</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Many who first hear about the above studies are asking a reasonable question: "What the hell?" </font><font style="vertical-align: inherit;">Why do scientists create chimeric killer viruses? </font><font style="vertical-align: inherit;">Politically correct answer: to develop preventive protection (vaccines and drugs) from possible natural chimeras and to understand the risks of their occurrence. </font><font style="vertical-align: inherit;">Here, in fact, that Barik and Shi Zhengli and co-authors themselves wrote on this subject in the same article of 2015:</font></font><br>
<blockquote><i>       ,                (GOF).     (Fig. 4a, b)   ,   SHC014-MA15,  ,    .  SHC014-MA15        SARS-CoV,  ,   CoV   Urbani     MA15,      ,    .  ,      Urbani-MA15 CoV, SHC014-MA15    (. 1).               ,    ,  ,       .                 ,             .          GOF;                    .         ,     ,  ,              .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">In addition to offering preparation against future emerging viruses, this approach must be considered in the context of the US government‚Äìmandated pause on gain-of-function (GOF) studies. On the basis of previous models of emergence (Fig. 4a,b), the creation of chimeric viruses such as SHC014-MA15 was not expected to increase pathogenicity. Although SHC014-MA15 is attenuated relative to its parental mouse-adapted SARS-CoV, similar studies examining the pathogenicity of CoVs with the wild-type Urbani spike within the MA15 backbone showed no weight loss in mice and reduced viral replication. Thus, relative to the Urbani spike‚ÄìMA15 CoV, SHC014-MA15 shows a gain in pathogenesis (Fig. 1). On the basis of these findings, scientific review panels may deem similar studies building chimeric viruses based on circulating strains too risky to pursue, as increased pathogenicity in mammalian models cannot be excluded. Coupled with restrictions on mouse-adapted strains and the development of monoclonal antibodies using escape mutants, research into CoV emergence and therapeutic efficacy may be severely limited moving forward. Together, these data and restrictions represent a crossroads of GOF research concerns; the potential to prepare for and mitigate future outbreaks must be weighed against the risk of creating more dangerous pathogens. In developing policies moving forward, it is important to consider the value of the data generated by these studies and whether these types of chimeric virus studies warrant further investigation versus the inherent risks involved.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Were these words prophetic? </font><font style="vertical-align: inherit;">At the end of 2014, the United States introduced a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">moratorium</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on state financing of such gain-of-function studies, but it </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">was canceled</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> almost immediately (in 2017) </font><font style="vertical-align: inherit;">. </font><font style="vertical-align: inherit;">And in China, no moratorium on such studies was introduced, and on the contrary, they opened new ‚Äúsuper (without) dangerous laboratories‚Äù of BSL-4 level, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">as in 2017 in Wuhan</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> :</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/ccd/f81/121/ccdf8112198fa480aebee48d16cb595d.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Just in case, I‚Äôll explain that until 2017, the laboratory at the Wuhan Institute of Virology was BSL-3, which was enough to work with coronaviruses. </font><font style="vertical-align: inherit;">Here are a couple of quotes from the above note on the creation of the Wuhan BSL-4 laboratory:</font></font><br>
<blockquote><i>    ‚Äî  ,   ,       BSL-4,           .      ;       ,  . ¬´   ¬ª.<br>
‚Ä¶<br>
      .   BSL-4     ;  ,  ,     ,        .<br>
<br>
 ,      ,      BSL-4     ‚Äî         ,   ,        ‚Äî   . ¬´       ,    ,   ¬ª, ‚Äî  .<br>
<br>
         BSL-4   .  ,             , ,   ,  .  ,    ,          .<br>
<br>
¬´        ¬ª, ‚Äî  .          ,    : ¬´  ,   ,   ¬ª.<br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Trevan says China‚Äôs investment in the BSL-4 laboratory can, above all, be a way to prove to the world that the country is competitive. </font><font style="vertical-align: inherit;">‚ÄúThis is a big status symbol in biology,‚Äù he says, ‚Äúregardless of whether they are needed or not.‚Äù</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Future plans include studying the pathogen that causes SARS, which also doesn‚Äôt require a BSL-4 lab, before moving on to Ebola and the West African Lassa virus, which do. Some one million Chinese people work in Africa; the country needs to be ready for any eventuality, says Yuan. ‚ÄúViruses don‚Äôt know borders.‚Äù<br>
‚Ä¶<br>
The plan to expand into a network heightens such concerns. One BSL-4 lab in Harbin is already awaiting accreditation; the next two are expected to be in Beijing and <b>Kunming, the latter focused on using monkey models to study disease.</b><br>
Lina says that China‚Äôs size justifies this scale, and that the opportunity to combine BSL-4 research with an abundance of research monkeys ‚Äî Chinese researchers face less red tape than those in the West when it comes to research on primates ‚Äî could be powerful. ‚ÄúIf you want to test vaccines or antivirals, you need a non-human primate model,‚Äù says Lina.<br>
But Ebright is not convinced of the need for more than one BSL-4 lab in mainland China. He suspects that the expansion there is a reaction to the networks in the United States and Europe, which he says are also unwarranted. He adds that governments will assume that such excess capacity is for the potential development of bioweapons.<br>
‚ÄúThese facilities are inherently dual use,‚Äù he says. The prospect of ramping up opportunities to inject monkeys with pathogens also worries, rather than excites, him: ‚ÄúThey can run, they can scratch, they can bite.‚Äù<br>
Trevan says China‚Äôs investment in a BSL-4 lab may, above all, be a way to prove to the world that the nation is competitive. ‚ÄúIt is a big status symbol in biology,‚Äù he says, ‚Äúwhether it‚Äôs a need or not.‚Äù</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Interestingly, in addition to Wuhan, they planned to open a new BSL-4 laboratory in Kunming, with an eye to testing vaccines on primates. </font><font style="vertical-align: inherit;">Kunming, I remind you, this is the capital of Yunnan. </font><font style="vertical-align: inherit;">It was in the nearby caves that Shi Zhengli found the very strains Rs3367 and RsSHC014. </font><font style="vertical-align: inherit;">By the way, primacy testing was mentioned as possible further steps for the development of preventive vaccines against potential future outbreaks of coronavirus in the ‚Äúsame‚Äù (how many times have I called it that?) Article by Barik and Shi Zhengli:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">However, further testing on nonhuman primates is required in order to translate these findings into pathogenic potential in humans. </font><font style="vertical-align: inherit;">It is important to note that the lack of available therapeutic agents determines the critical need for further study and development of treatment methods. </font><font style="vertical-align: inherit;">With this knowledge, surveillance programs, diagnostic reagents, and effective treatments can be developed that can protect against the appearance of group 2b-specific coronaviruses, such as SHC014, and can be applied to other coronavirus branches that support similar heterogeneous pools.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">However, further testing in nonhuman primates is required to translate these finding into pathogenic potential in humans. Importantly, the failure of available therapeutics defines a critical need for further study and for the development of treatments. With this knowledge, surveillance programs, diagnostic reagents and effective treatments can be produced that are protective against the emergence of group 2b‚Äìspecific CoVs, such as SHC014, and these can be applied to other CoV branches that maintain similarly heterogeneous pools.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">It is possible that by 2019 the creation and testing of potential vaccines from various SARS-like coronaviruses was already in full swing.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">About how many wonderful epidemics the enlightenment spirit prepares ...</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Well then, let's take a look at the lab leak version. To begin with, I‚Äôll give a brief historical background about other leaks. Because shoots of viruses from laboratories in the past happened more than once. First of all, of the same SARS-CoV: for the first time he fled </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in the summer of 2003</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in Singapore, then </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in December 2003</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in Taiwan, and </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in the spring of 2004 he</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> twice flew to Beijing. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
There were alarming calls in Europe and the USA, although there were no infections there. For example, in France, a laboratory somehow </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">lost test tubes with SARS</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , and in the USA BSL-4 laboratory in Texas, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">test tubes</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> with the Venezuelan hemorrhagic fever virus were </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">missing</font></a><font style="vertical-align: inherit;"> :</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Only one scientist worked with the virus, and Reyes said the lab suspects the scientist accidentally threw a test tube in November. </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
... </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The Galveston Biolaboratory has the most stringent security measures, because it is studying BSL-4 biosafety materials, that is, dangerous infectious diseases that do not have vaccines or drugs. </font><font style="vertical-align: inherit;">BSL-4 materials include Guanarito, Ebola, and smallpox.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Only one scientist worked with the virus, and Reyes said the lab suspects that scientist accidentally threw the vial away in November. <br>
‚Ä¶<br>
Galveston biolab requires the most stringent safety measures because it studies biosafetly level BSL-4 materials, or dangerous infectious diseases that have no vaccines or cures. BSL-4 materials include Guanarito, Ebola and smallpox.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">History knows other, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">much larger-scale leaks</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">For example, the "resurrection" of the H1N1 flu virus in 1977, which was previously considered to have disappeared. </font><font style="vertical-align: inherit;">Yes, this is the virus of the same "Spanish woman":</font></font><br>
<blockquote><i>   H1N1    1918   ,      (    1947 ),   1957    ¬´¬ª  H2N2,   .   H1N1, -, ,      20 .  1969   H3N2     H2N2   .<br>
<br>
  1977 .   H1N1         ,    1978 .  ,    1977 .    H1N1  - ,  ,     .    ,    ,  ,   H1N1 1977        H1N1,   1949‚Äì1950 ,    ,     .<br>
‚Ä¶<br>
   2009‚Äì2010 .      ,    H1N1  1977    : ¬´      ‚Äî      A H1N1,         1977       ¬ª.<br>
‚Ä¶<br>
  ,   1977 .        H1N1,  ,             ¬´ ¬ª (Kung 1978).   ,  ,  ,   1970-    (   )         .             .<br>
<br>
,    ,     1977‚Äì78 .     -   ,     , -,     H1N1  1978 .   H1N1,         ,       .  1976‚Äì77     20        H1N1  1957,       .    1977         ,       .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Human influenza H1N1 viruses appeared with the 1918 pandemic, and persisted, slowing accumulating small changes in its genome (with a major change in 1947), until the H2N2 ‚ÄúAsian‚Äù flu appeared in 1957, causing a worldwide pandemic. H1N1 influenza virus then apparently became extinct, and was not isolated for 20 years. In 1969 the ‚ÄúHong Kong‚Äù H3N2 virus replaced the H2N2 virus, and is still circulating.<br>
In September 1977 an H1N1 influenza virus was isolated from human infections in the Far East region of the Soviet Union, and in early 1978 the Chinese reported they had isolated H1N1 virus in May of 1977 in northeast China adjacent to the Soviet outbreak. Using the early genetic tools available at the time, the 1977 H1N1 virus was found to be closely related to H1N1 human influenza viruses circulating in 1949‚Äì1950, but not to those circulating earlier or later.<br>
‚Ä¶<br>
Only since 2009‚Äì2010 did major papers begin to state directly the 1977 emergence of H1N1 influenza was a laboratory related release: ‚ÄúThe most famous case of a released laboratory strain is the re-emergent H1N1 influenza A virus which was first observed in China in May of 1977 and in Russia shortly thereafter.‚Äù <br>
‚Ä¶<br>
The speculation that the 1977 release may have been related to H1N1 vaccine research is supported by the observation that in the initial outbreaks in China, nine of the ten viral isolates expressed ‚Äútemperature sensitivity‚Äù (Kung 1978). Temperature sensitivity normally an uncommon trait, but one that was in the 1970s (and still is) a fundamental trait for making live attenuated influenza vaccines. Temperature sensitivity generally occurs only after a series of substantial laboratory manipulations and selections.<br>
Interestingly, further investigation indicated the circulating strains in 1977‚Äì78 were often comprised of mixed temperature-sensitive and normal components, and that temperature sensitivity apparently disappeared from the post-1978 H1N1 lineage rapidly. Escape of a mid-protocol population of H1N1 virus undergoing laboratory selection for temperature sensitive mutants would provide such a mixed population. In 1976‚Äì77 laboratory personnel in their late teens or early 20s would not have been exposed to pre-1957 H1N1 influenza viruses, and been susceptible to laboratory infections. The low severity of the 1977 pandemic might be in part due to the temperature sensitivity of the virus, a trait that limits virus replication in pulmonary tissues.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">It seems that the creation of temperature-sensitive viral mutants to develop potential ‚Äúattenuated‚Äù vaccines was widespread at the end of the twentieth century. If you remember, in 1990, Barik </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">also experimented</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> with the creation of temperature-sensitive strains.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Could something like this cause the coronavirus pandemic of 2019? Why not? Here several options are possible - from developing a potential vaccine to just research on laboratory recombination of the bat and pangolin viruses. Some particularly ambitious researcher could even decide on personal initiative to combine the two ‚Äúfashion themes‚Äù - adding a furin site and transplanting RBM from a strain of one species (pangolin) to another (bats), so that later, confirming the increased virulence of the new chimeric design, to release another formidable article about the danger that awaits humanity in Yunnan caves or in wet markets. And if a vaccine against such a dangerous strain could have been preventively created, then honor and respect for such a researcher would have been guaranteed.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Do I affirm that it was like that? Of course not. Because today there is no evidence of such a scenario. There is only a series of strange coincidences - for example, that the outbreak of the Yunnan coronavirus occurred thousands of kilometers from Yunnan in the very market that is closest to the Wuhan Institute of Virology. Or maybe not on the market, because of the first 4 sick patients, three </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">were not</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on the market </font><font style="vertical-align: inherit;">. Well, the coincidence in structural features of the genome of the virus, which resemble those manipulations that virologists have repeatedly carried out with such viruses in the laboratory. But coincidences are not proof.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Moreover, coincidences do occur, and, of course, such a strain could also have arisen in nature. It‚Äôs not yet clear how exactly - for this, the bat and pangolinium strains should meet in one cell (the cell of someone‚Äôs body, not the metal one), and in Wuhan, since the outbreak occurred there (otherwise we would have seen other foci of Covid by the path of the first livestock to Wuhan). Given that bats </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">were not sold</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in the Wuhan market </font><font style="vertical-align: inherit;">, and generally hibernated at this time of the year, this option is still an unsolved mystery.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, recently there was news that in 2018, American experts carried out an inspection of the Wuhan Institute by virologists, and even talked with Shi Zhengli. </font><font style="vertical-align: inherit;">The result of their "tour" were two diplomatic telegrams to Washington, in which they noted a number of weaknesses in ensuring the safety of the laboratory:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Telegram sources said they should have raised the alarm about serious security issues at the WIV lab, especially regarding its work with bat coronaviruses. </font><font style="vertical-align: inherit;">Embassy officials called for the United States to pay more attention to this laboratory and provide additional support. </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
...</font></font><br>
¬´       WIV  ,              ,          ¬ª, ‚Äî     19  2018 ,        ,    ,      WIV. (        .)<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Chinese researchers at WIV received help from the Galveston National Laboratory at the Medical Department of the University of Texas and other organizations in the United States, but the Chinese sought additional help. </font><font style="vertical-align: inherit;">Telegrams argue that the United States should provide further support to the Wuhan laboratory, mainly because their study of bat coronaviruses was important, but also dangerous.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">Sources familiar with the cables said they were meant to sound an alarm about the grave safety concerns at the WIV lab, especially regarding its work with bat coronaviruses. The embassy officials were calling for more U.S. attention to this lab and more support for it, to help it fix its problems.<br>
‚Ä¶<br>
‚ÄúDuring interactions with scientists at the WIV laboratory, they noted the new lab has a serious shortage of appropriately trained technicians and investigators needed to safely operate this high-containment laboratory,‚Äù states the Jan. 19, 2018, cable, which was drafted by two officials from the embassy‚Äôs environment, science and health sections who met with the WIV scientists. (The State Department declined to comment on this and other details of the story.)<br>
The Chinese researchers at WIV were receiving assistance from the Galveston National Laboratory at the University of Texas Medical Branch and other U.S. organizations, but the Chinese requested additional help. The cables argued that the United States should give the Wuhan lab further support, mainly because its research on bat coronaviruses was important but also dangerous.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">It is funny that the Texas laboratory in Galveston, which at one time </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">itself had lost a</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> test tube with the Venezuelan virus </font><font style="vertical-align: inherit;">, helped the Wuhan people </font><font style="vertical-align: inherit;">. </font><font style="vertical-align: inherit;">Moreover, she helped not only in words, Wuhan specialists did take training there, which was even written by Wuhan Bulletin (though, now the publication has been deleted from the site, but it is still </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=https://web.archive.org/web/20200404101918/" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">available</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on the web archive </font><font style="vertical-align: inherit;">):</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/64b/2e0/bb3/64b2e0bb3a8a447ed5bc0805f0d8e403.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The final touch to the family portrait of laboratory leaks: in November 2019, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">an outbreak of brucellosis</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (a bacterial infection) </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">occurred</font></a><font style="vertical-align: inherit;"> in two research centers in Lanzhou </font><font style="vertical-align: inherit;">, affecting more than 100 researchers working there.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Possible traces of man-made</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Then leave the virologists alone and turn our eyes again to the virus itself. Are there any obvious signs of man-madeness in it? First, a few words about what ‚Äúexplicit‚Äù means. It is clear that in nature any mutations can occur completely by accident. Even if the tie-in that created the furin site in CoV2 was not ‚ÄúPRRA‚Äù, but ‚ÄúMADEINWVHANPRRA‚Äù, there would still be a non-zero chance that it could have arisen. But for us, and for any court, I think this would be enough to prove the man-made origin </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">beyond a reasonable doubt</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> .</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The main problem with such evidence is that even in a man-made virus they simply may not exist. Roughly speaking, a good genetic engineer can create a synthetic virus "identical to natural." Moreover, often researchers deliberately introduce synonymous mutations into their structures so that their strain and the natural one can be distinguished later. But if the creator of the virus does not reveal these markers of man-madeness, it is impossible to distinguish them from natural mutations. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But sometimes traces can remain, especially if the creators do not try to hide the man-madeness of their design. First of all, we are talking about the places of DNA cuts (I recall that manipulations with RNA viruses are carried out precisely in their </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">DNA constructs</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">), necessary for creators to stitch different segments of the genome or to cut out old and insert new sites. Indeed, DNA can be cut not in arbitrary places (Crisper doesn‚Äôt count), but only where the sequence of nucleotides (usually 4-6 ‚Äúletters‚Äù) coincides with the sequence recognized by one or another </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">restriction enzyme</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , that is, an enzyme that breaks down nucleotide chains . Moreover, such an analysis complicates the fact that there are hundreds of different types of restriction enzymes used in genetic engineering. But let's try to conduct such an analysis for CoV2. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
To begin with - an example of the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">work</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> of the Barik group from 2008, where they took Bat-SCoV and replaced RBD in its spike protein with RBD from human SARS. Here's how they describe the creation of their chimera:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/8b7/a2d/fa4/8b7a2dfa4af96ed4649eec6b53a96a84.png"></div><blockquote><i><b>   SARS-CoV  Bat-SCoV. <br>
</b>(A)    SARS-CoV  Bat-SCoV (  GenBank FJ211859)    . ()       nsp   ORF1ab ( , - ;  , nsp5 [3C- ]).    ,     ,   A  F. ,    Bat-SCoV,     SARS-CoV,   ,  Bat-SCoV  2 , Bat-E1  Bat-E2,    SARS-E.<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text"><b>Schematic representation of SARS-CoV and Bat-SCoV variants. </b><br>
(A) Schematic representation of SARS-CoV and Bat-SCoV (GenBank accession no. <a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow">FJ211859</a>) genomes and reverse genetics system. (Top) Arrowheads indicate nsp processing sites within the ORF1ab polyprotein (open arrowheads, papain-like proteinase mediated; filled arrowheads, nsp5 [3C-like proteinase] mediated). Immediately below are the fragments used in the reverse genetics system, labeled A through F. The fragments synthesized to generate Bat-SCoV exactly recapitulate the fragment junctions of SARS-CoV with the exception that the Bat-SCoV has 2 fragments, Bat-E1 and Bat-E2, which correspond to the SARS-E fragment.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">As you can see, first the Barik group created a synthetic clone of the bat Bat-SCoV, and moreover, according to the ‚Äúpatterns‚Äù of the synthetic clone SARS-CoV that they had previously created. That is, for the bat clone, they used the same 6 segments with the same restriction enzyme sites that they had previously used for SARS-CoV, which allowed them to swap virus segments between different strains like Lego pieces. Here is a detailed description of creating a chimeric mutant:</font></font><br>
<blockquote><i>,  -    ,       SARS-CoV (24, 33, 53)   . ,   Bat-F           A-E SARS-CoV   F Bat-SCoV,    Bgl-NotI.  Bat-SCoV  Bat-SRBD, Bat-SRBM  Bat-Hinge <b>,  7    Bat-SCoV,     BglI  Bat-A, Bat-B, Bat-C  Bat -D, BglI  AflII  Bat-E1  Bat-E2  BglI  NotI  Bat-F.</b> ,     .      m7Message mMachine (Ambion),      Vero (24, 53).<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Viruses containing PCR-generated insertions within the viral coding sequence were produced by using the SARS-CoV assembly strategy (24, 33, 53) with the following modifications. Briefly, for Bat-F virus, full-length cDNA was constructed by ligating restriction products from SARS-CoV fragments A‚ÄìE and Bat-SCoV fragment F, which required a BglI-NotI digestion. For Bat-SCoV and Bat-SRBD, Bat-SRBM, and Bat-Hinge, <b>plasmids containing the 7 cDNA fragments of the Bat-SCoV genome were digested by using BglI for Bat-A, Bat-B, Bat-C, and Bat-D, BglI and AflII for Bat-E1 and Bat-E2, and BglI and NotI for Bat-F.</b> Digested, gel-purified fragments were simultaneously ligated together. Transcription was driven by using a T7 mMessage mMachine kit (Ambion), and RNA was electroporated into Vero cells (24, 53).</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">All these three-letter abbreviations (BglI, AflII, NotI, etc.) in the sentence highlighted above are different types of restriction enzymes. </font><font style="vertical-align: inherit;">Let's see if there will be any differences in the restriction enzyme sites in the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">genome</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (spike protein) of the chimera compared to the genome of the original SARS-CoV:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/lk/yw/r5/lkywr5wi9t3z6g3bj5phgbnhu9y.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As can be seen, the restriction enzyme sites of the chimera are almost identical to those pieces of the original sequences in Bat-SCoV or SARS from where they were taken. </font><font style="vertical-align: inherit;">The only differences are visible at the crosslinking sites of the inserted SARS piece. </font><font style="vertical-align: inherit;">Here, for example, the left (5'-) edge of the insert:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/jd/m_/aw/jdm_awiat-0-e6tstpm2gk1xjam.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Here Bat-SCoV and SARS turned out to have a common identical nucleotide region (the intersection of the turquoise and pink regions), and there are no new restriction enzyme sites at the place of the two sequences stitching, but on the contrary the SspI site from SARS disappeared. </font><font style="vertical-align: inherit;">And here is the right (3'-) edge of the insert:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/ix/dw/wd/ixdwwdufidplsqru-5o581lapom.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Here, on the contrary, all the old restriction enzyme sites remained in the bonding place, and even new ones appeared, for example, </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">EcoR</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> II. If I did not know that the chimeric genome is the result of man-made manipulations, could I understand this by looking at these 3 sequences? It is unlikely that even if some suspicions had crept in, then it was certainly not </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">beyond a reasonable doubt</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . Perhaps, specialists in genetic engineering could have seen this for some other signs, I can‚Äôt presume to say - I‚Äôll be glad of any educational program here. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But in any case, let's compare the spike protein in RaTG13, CoV2 and pangolin-19. Suddenly, something will jump on us as it jumps out! </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
This is what RBD (highlighted in light green) and RBM (yellow) look like for all three:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/z5/oz/7y/z5oz7yyz_nxu5qmwa7mgnnxz2ci.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
What's so interesting? It was interesting to see the </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">EcoR</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> I </font><font style="vertical-align: inherit;">site </font><font style="vertical-align: inherit;">on the 5'-edge of RBM for all three (the first joint between the green and yellow areas) - very convenient. I wonder how common is this feature in other strains? A cursory analysis showed that our trinity are champions in the number of such sites in the genome, in other bat strains there are only 5 of them:</font></font><br>
<br>
<div style="text-align:center;"><img width="400" src="https://habrastorage.org/getpro/habr/post_images/44e/fd4/e4e/44efd4e4eda10d4ac61bc798b91a2259.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Returning to the analysis of RBD, from the features of CoV2, new restriction enzyme sites highlighted in red rectangles can be noted - they coincide with unique mutations in the amino acid sequence (also marked with red rectangles on amino acid sequences in the far right column). Just in case, I highlighted several more new sites: blue rectangles and a green rectangle, which is located in the region of the only amino acid that differs between RBM CoV2 and pangolin-19. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Let's compare in all three strains the place of the PRRA insert, which created the furin site in CoV2:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/webt/tg/1x/-n/tg1x-nfgwm-4hcs0oule3ryarbs.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Here, too, several new sites appeared (highlighted in blue) on both sides of the new insert. </font><font style="vertical-align: inherit;">Could they be used to create a furin site? </font><font style="vertical-align: inherit;">Theoretically, yes. </font><font style="vertical-align: inherit;">But the insertion could be done using existing sites, or even creating segments with new sites, which are then combined seamlessly, that is, without creating new sites at the junction. </font><font style="vertical-align: inherit;">If you remember, Barik back in 2002 </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">applied this technology</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> to create a synthetic clone of mouse coronavirus:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">The junction sites of the restriction sites that are located at the ends of each cDNA are systematically removed during assembly of the complete full-sized cDNA product, which allows reassembly without modification of the nucleotides.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">The interconnecting restriction site junctions that are located at the ends of each cDNA are systematically removed during the assembly of the complete full-length cDNA product, allowing reassembly without the introduction of nucleotide changes.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">And in 2003 he </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">repeated</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on the synthetic clone SARS-CoV:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">To quickly assemble consensus clones, we used class IIS restriction endonucleases, which are cut in asymmetric regions and leave asymmetric ends. </font><font style="vertical-align: inherit;">These enzymes generate specific unique overhangs that </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">provide seamless ligation of two cDNAs with subsequent loss of the restriction site</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> .</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text">To rapidly assemble consensus clones, we used class IIS restriction endonucleases that cut at asymmetric sites and leave asymmetric ends. These enzymes generate strand-specific unique overhangs<b> that allow the seamless ligation of two cDNAs with the concomitant loss of the restriction site.</b></div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Today, technology for juggling gene sequences is already so automated and put on stream that in a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Chinese article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> from October 2019 about inserting a new furin site into chicken coronavirus, there are only a couple of suggestions for its description:</font></font><br>
<blockquote><i><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">2.2. </font><font style="vertical-align: inherit;">Creation of a recombinant virus The</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
recombinant virus rYN-S2 / RRKR, containing a spike protein with a furin site S2 ', was obtained by vaccinia recombination, as described previously [20, 28]. </font><font style="vertical-align: inherit;">Briefly, </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">a S ‚Ä≤ furin site plasmid was generated using the Seamless Assembly kit</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (Invitrogen, Carlsbad, CA, USA) and transfected into CV-1 cells infected with vaccinia virus containing the YN-? S-GPT gene. </font><font style="vertical-align: inherit;">The Furin-S2 site was introduced into YN cDNA by homologous recombination using a temporal dominant selection system [25].</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text"><b>2.2. Generation of Recombinant Virus</b><br>
Recombinant rYN-S2/RRKR virus containing an S protein with the furin-S2? site was generated by vaccinia recombination, as described previously [20,28]. Briefly, <b>plasmid with the furin-S2? site was generated using the Seamless Assembly kit</b> (Invitrogen, Carlsbad, CA, USA) and transfected into CV-1 cells infected by vaccinia virus containing the genome of YN-?S-GPT. Furin-S2‚Äô site was introduced into the YN cDNA by homologous recombination using the transient dominant selection system [25].</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">It‚Äôs impossible not to admire what progress has come to! </font><font style="vertical-align: inherit;">Here is a description of the above </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Seamless Assembly kit</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> :</font></font><br>
<blockquote><i>      GeneArt       1  4  ,    ,       <b> 30-    </b>.             E.coli      .<br>
‚Ä¢    ‚Äî   4          ( 13 );    ,   <br>
‚Ä¢    ‚Äî    ,   ,   ,       90%     .<br>
‚Ä¢   ‚Äî         <br>
‚Ä¢   ‚Äî  -        -,         <br>
‚Ä¢   ‚Äî            , ,     .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">The GeneArt Seamless Cloning and Assembly Kit enables the simultaneous and directional cloning of 1 to 4 PCR fragments, consisting of any sequence, into any linearized vector, <b>in a single 30-minute room temperature reaction</b>. The kit contains everything required for the assembly of DNA fragments, and their transformation into E. coli for selection and growth of recombinant vectors.<br>
‚Ä¢ <b>Speed and Ease?</b>‚Äî?Clone up to 4 DNA fragments, with sequence of your choice, simultaneously in a single vector (up to 13 Kb); no restriction digestion, ligation or recombination sites required<br>
‚Ä¢ <b>Precision and Efficiency</b> ‚Äî Designed to let you clone what you want, where you want, in the orientation you want, and achieve up to 90% correct clones with no extra sequences left behind<br>
‚Ä¢ <b>Vector Flexibility</b> ‚Äî Use our linear vector or a vector of your choice<br>
‚Ä¢ <b>Free Tools</b> ‚Äî Design DNA oligos and more with our free web-based interface that walks you step-by-step through your project<br>
‚Ä¢ <b>Diverse Applications</b> ‚Äî Streamline many synthetic biology and molecular biology techniques through the rapid combination, addition, deletion, or exchange of DNA segments</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Up to 4 DNA fragments can be glued in the desired sequence in just half an hour, without a headache with restriction enzymes or ligation. </font><font style="vertical-align: inherit;">And upload your creation to </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">E. coli</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> to propagate the resulting design. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Summarizing the analysis of restriction enzyme sites, it must be recognized that no definite conclusions can be drawn from its results. </font><font style="vertical-align: inherit;">Unless once again it was possible to make sure that not only CoV2 is unique, but RaTG13 itself is a very unusual comrade, and that it is worth further studying his biography.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Codon Preferences</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
For these purposes, I also decided to take a look at </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">codon usage bias</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> to look at which strains of other viruses look like CoV2 and RaTG13. </font><font style="vertical-align: inherit;">It is no secret that viruses tend to adapt in their </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">codon signature</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> to the preferences of their hosts, so I expected to see a similar pattern with other hepatomirus viruses in RaTG13, and also hoped to see a difference from pangolin strains. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Here SARS-CoV, for example, is very similar to Rs3367 and RsSCH014, as you might expect:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/6b6/f89/f3c/6b6f89f3c6569134e023778e2d81639a.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Between themselves, by the way, SARS, MERS and CoV2 differ:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/bd5/fd6/322/bd5fd6322db1e9a50b41b9a78cdc01e5.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
RaTG13 is similar to CoV2, which is also to be expected:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/388/c32/4bd/388c324bd993aeaea221a7fb98f036e4.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But RaTG13 is really not super similar to the pangolin strains, and the pangolin strains are not exactly identical to each other:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/d3b/cf0/916/d3bcf09160ac657d756d56eadabb5036.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
On the ZXC21 and ZC45 is also not very similar:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/c6b/df0/348/c6bdf03486c3d1564a3fc6c2da45789b.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Of the Yunnan strains, RaTG13 from Rs3367 and RsSCH014 is quite far, and closest to LYRa11, but also with noticeable differences:</font></font><br>
<br>
<div style="text-align:center;"><img width="600" src="https://habrastorage.org/getpro/habr/post_images/c5b/761/ec2/c5b761ec2455244434c5673157f4e74a.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In general, as always, RaTG13 and CoV2 are somehow separate. </font><font style="vertical-align: inherit;">I was also intrigued by the AAA codon - they use it much more often than their fellow tribesmen:</font></font><br>
<br>
<div style="text-align:center;"><img width="400" src="https://habrastorage.org/getpro/habr/post_images/65f/5f6/d83/65f5f6d832d5f7fba40bdc4380d03147.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
This is probably just another coincidence, but a similar proportion between AAA and AAG </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">is observed</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">E. coli</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">Can the cDNA signature of cDNA change as it is cultivated for a long time in cell culture? </font><font style="vertical-align: inherit;">In theory, yes, but I haven‚Äôt dug this topic deeply yet. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Well, codon analysis also did not reveal any obvious signs of creativity, but once again confirmed the uniqueness of CoV2 and RaTG13. </font><font style="vertical-align: inherit;">What do we have in a dry state? </font><font style="vertical-align: inherit;">So far, only a certain set of strange coincidences, which, as scientists like to say, </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">taken together</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , that is, in the aggregate, makes you think very hard. </font><font style="vertical-align: inherit;">And certainly it does not allow rejecting the hypothesis about the man-made nature of CoV2.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">But what about the refutation of man-madeness in Nature?</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
How does not allow? And why does the author of </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">that article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in Nature allow? In fact, there is no </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">refutation of</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> man-madeness in that article. There is only a loud ‚Äúwe don‚Äôt think so‚Äù, based on a very unsteady foundation. Judge for yourself - these are the main points of the authors in support of the miraculous:</font></font><br>
<blockquote><i>    ,  SARS-CoV-2     ACE2   ,   ,      ,     RBD       SARS-CoV [ SARS ‚Äî ..]    .  ,     SARS-CoV-2   ACE2,  ,          ACE2,        .    ,  SARS-CoV-2     .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">While the analyses above suggest that SARS-CoV-2 may bind human ACE2 with high affinity, computational analyses predict that the interaction is not ideal and that the RBD sequence is different from those shown in SARS-CoV to be optimal for receptor binding. Thus, the high-affinity binding of the SARS-CoV-2 spike protein to human ACE2 is most likely the result of natural selection on a human or human-like ACE2 that permits another optimal binding solution to arise. This is strong evidence that SARS-CoV-2 is not the product of purposeful manipulation.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">This citation in the article is shown right below the diagram showing the identical RBMs of CoV2 and pangolin-19. Wait, then what does "computer simulation" have to do with it? The most likely man-made scenario is the transfer of RBM from a strain of one animal to a strain of another - which virologists have already done repeatedly. Therefore, the logical chain of authors does not stand up to criticism: ‚Äúon a computer it was possible to design a steeper virus, so CoV2 is the result of natural selection. Oh, so this is strong evidence that CoV2 is not made by hands! ‚Äù Apparently, the authors have bad logic. Their further theses confirm this:</font></font><br>
<blockquote><i> ,      , ,         ,   -.   ,    ,  SARS-CoV-2    -    .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">Furthermore, if genetic manipulation had been performed, one of the several reverse-genetic systems available for betacoronaviruses would probably have been used. However, the genetic data irrefutably show that SARS-CoV-2 is not derived from any previously used virus backbone.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Again, the same logical mistake, veiled by loud turns: ‚Äúgenetic analysis irrefutably proves that CoV2 was definitely not created on the basis of previously known viruses!‚Äù Well thanks, Captain Evidence. And what, really, the creators of the virus could not make cDNA backbone from a previously unpublished strain of the virus - for example, from the same RaTG13? Yes Easy. Also, it would not be difficult for them to then insert the RBM pangolinium and the furin site there. Virologists have been doing this for 20 years, and modern genetic engineering tools make such manipulations accessible even to students. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As for the origin of the furin site in cell culture, the authors also express strange theses:</font></font><br>
<blockquote><i>    ,    O-        .                in vitro  in vivo.  ,   SARS-CoV-2            -     , <b>   </b>.                 ACE2,     ,    <b>    </b>.<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">The acquisition of both the polybasic cleavage site and predicted O-linked glycans also argues against culture-based scenarios. New polybasic cleavage sites have been observed only after prolonged passage of low-pathogenicity avian influenza virus in vitro or in vivo. Furthermore, a hypothetical generation of SARS-CoV-2 by cell culture or animal passage would have required prior isolation of <b>a progenitor virus with very high genetic similarity, which has not been described. </b>Subsequent generation of a polybasic cleavage site would have then required repeated passage in cell culture or animals with ACE2 receptors similar to those of humans, but such work has also <b>not previously been described</b>.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Firstly, the authors themselves previously provide links to works where a furin site arose as viruses were cultured in cells. And secondly, what does it mean that a similar strain of the virus has not been described - but what about RaTG13? If RBM was synthetically replaced with pangolinium in it, and then the chimeric strain was cultured in cell culture, then the furin site could well appear both in this way and in addition, in the same way, the new strain could acquire other mutations that distinguish CoV2 from RaTG13 and pangolin-19 . </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But in terms of precisely the man-made variant of the appearance of the furin site, I more likely see the option with a targeted insert - as in another Chinese </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">work from October 2019</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> with chicken coronavirus. And then the created strain could acquire new mutations as it is cultivated </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in vitro</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">or </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in vivo</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> as a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">murine strain of</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> MA15 in 2007, for example.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Shi Zhengli 2020</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
While I was writing this article, the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">work of a</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> large team of Chinese virologists, including Shi Zhengli, in which in February they tested against CoV2 their many years of development </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">from many types of coronaviruses</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> - a peptide that is designed to block the fusion of a spike-like protein with a cell membrane </font><font style="vertical-align: inherit;">, came out </font><font style="vertical-align: inherit;">. </font><font style="vertical-align: inherit;">The authors, of course, mention the new furin site of CoV2, and suggest that it can play an important role in the much more efficient penetration of CoV2 into the cell:</font></font><br>
<blockquote><i>    ,  SARS-CoV-2        ,  SARS-CoV,   ,    SARS-CoV-2        .<br>
‚Ä¶<br>
 ,  Œ≤-     S1/S2,         . , SARS-CoV           ,        L  .    S1/S2       SARS-CoV    .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text">In this study, we have shown that SARS-CoV-2 exhibits much higher capacity of membrane fusion than SARS-CoV, suggesting that the fusion machinery of SARS-CoV-2 is an important target for development of coronavirus fusion inhibitors. <br>
‚Ä¶<br>
Generally, Œ≤-B coronaviruses lack the S1/S2 furin-recognition site, and their S proteins are uncleaved in the native state. For example, SARS-CoV enters into the cell mainly via the endosomal membrane fusion pathway where its S protein is cleaved by endosomal cathepsin L and activated. Inducing the S1/S2 furin-recognition site could significantly increase the capacity of SARS-CoV S protein to mediate cellular membrane surface infection.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">In this context, it becomes interesting, but have the authors previously conducted some experiments on how the addition of new furin sites to various coronaviruses can influence the effectiveness of their peptide? But we will not build unnecessary guesses, let time put everything in its place. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, Barik, apparently, decided to keep up with Shi Zhengli, and also </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">joined</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> the race to find funds from CoV2. As I understand it, he and co-authors took the data they already had on the effectiveness of their nucleoside analogue (Œ≤-D-N4-hydroxycytidine, NHC) against SARS-CoV and MERS, added </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in vitro</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> data on CoV2, and sent it to print. Nucleoside analogues (such as the famous </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">remdesivir</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">) Is a fundamentally different approach than that of Shi Zhengli and co-authors: here the authors are trying to prevent the virus from replicating by slipping ‚Äúdefective‚Äù letters of the genetic alphabet, and Shi Zhengli and co-authors are trying to prevent the virus from entering the cell. </font><font style="vertical-align: inherit;">Theoretically, these approaches can be combined.</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">This is the end, beautiful friend</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Oh, I hope until this moment at least someone reads. Sorry if I bore you. Himself in shock: the rabbit hole turned out to be a whole underground kingdom. I also hope that it was interesting for you to plunge into the world of virology and to open-mindedly consider the hypothesis of the man-made nature of CoV2. In my opinion, the dataset that I have cited, in aggregate, does not allow rejecting this hypothesis. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Just in case, I‚Äôll explain: this does NOT mean that CoV2 was precisely synthesized in the laboratory. Yes, purely technically, it would not be difficult for a modern virologist to create such a strain. But there is no direct evidence that anyone did this. And strange coincidences cannot yet serve even as indirect evidence.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But the inverse hypothesis about the exclusively natural nature of the virus is also not yet supported by solid evidence. So far, no intermediate ancestors have been found between RaTG13, pangolin-19, and CoV2, in which it would be possible to trace the selective recombination that we observe in CoV2, the question of its origin remains open. Perhaps </font><font style="vertical-align: inherit;">no one </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">speaks</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> better than Ralph Barik himself </font><font style="vertical-align: inherit;">:</font></font><br>
<blockquote><i><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Which animals are zoonotic carriers of SARS-CoV-2? </font></font><br>
</b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Such animals have not yet been found. </font><font style="vertical-align: inherit;">There is evidence that pangolins could potentially be an intermediate host, but pangolin viruses are only 88‚Äì98% identical to SARS-CoV-2. </font><font style="vertical-align: inherit;">For comparison, strains of civet and raccoon coronaviruses of the first SARS were 99.8% identical to 2003 SARS-CoV strains. </font><font style="vertical-align: inherit;">In other words, we are talking about several mutations between strains of civet, raccoon and humans in 2003. </font><font style="vertical-align: inherit;">Pangolins [CoV2 strains] have more than 3,000 nucleotide changes, so pangolins can in no way be a reservoir species. </font><font style="vertical-align: inherit;">Absolutely no chance.</font></font><br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Source text</font></font></b>
                        <div class="spoiler_text"><b>What is the reservoir species of SARS-CoV-2?<br>
</b>They have not identified the actual reservoir species. Reports show that pangolins are potentially the intermediate host, but pangolin viruses are 88‚Äì98% identical to SARS-CoV-2. In comparison, civet and racoon dog strains of SARS coronaviruses were 99.8% identical to SARS-CoV from 2003. In other words, we are talking about a handful of mutations between civet strains, racoon dog strains and human strains in 2003. Pangolin [strains of CoV2] have over 3000 nucleotide changes, no way they are the reservoir species. Absolutely no chance.</div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">So it goes. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
****************************************.... ***************************</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">UPD: about 4% of the difference between the genomes of RaTG13 and Cov2</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Some critics of the laboratory hypothesis have argued that the observed ~ 4% genetic difference between RaTG13 and CoV2 is too large for this to happen in the laboratory if RaTG13 itself was used as the basis. The observed mutation rates for RNA viruses </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">vary</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> widely - from 10 </font></font><sup><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">-6</font></font></sup><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> to 10 </font></font><sup><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">-4</font></font></sup><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> nucleotides per </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in </font></font></i><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">vitro</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> replication </font><font style="vertical-align: inherit;">, and in humans, CoV2 appears to mutate at a rate of 25 mutations per year. Thus, according to the logic of critics, it would take years, if not decades, for these two strains to diverge by 4%. Despite the fact that this is a reasonable objection, I see several problems with it. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
First, </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in </font></font></i><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">vitro</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> mutation rates are </font><font style="vertical-align: inherit;">much higher than</font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in </font></font></i><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">vivo</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , as you can passage cells much more often than infect new animals. As shown by experiments with SARS and MERS </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in </font></font></i><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">vitro</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , significant mutations can be observed after several passages. For example, in </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">a</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 2004 </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">article</font></a><font style="vertical-align: inherit;"> it was reported that after 600 passages, 2.1% differences in the genomic sequences of the spike-like proteins between the original strain and its offspring were already observed: </font></font><br>
<br>
<img width="700" src="https://habrastorage.org/webt/zi/o4/5q/zio45qtuufw9dedgkplfh84auas.png"><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Moreover, in the presence of some antiviral drugs, for example, the same nucleoside analogues (ribavirin or remdesivir ), the frequency of mutations in RNA viruses can </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">increase</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> threefold:</font></font><br>
<blockquote><i>      . 10-4     ,        -. <b>             ,        6 .</b>         in vitro  ,       ,   ,      .<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"> </b>
                        <div class="spoiler_text"> We obtained an estimate of the spontaneous mutation rate of ca. 10<sup>-4</sup> substitutions per site or lower, a value within the typically accepted range for RNA viruses. <b>A roughly threefold increase in mutation rate and a significant shift in mutation spectrum were observed in samples from patients undergoing 6 months of interferon plus ribavirin treatment.</b> This result is consistent with the known in vitro mutagenic effect of ribavirin and suggests that the antiviral effect of ribavirin plus interferon treatment is at least partly exerted through lethal mutagenesis. </div>
                    </div></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Thus, if the laboratory ancestor of CoV2 was tested to assess how its mutagenesis can change the effectiveness of potential vaccines or antiviral drugs against it, it could very quickly accumulate significant differences. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But perhaps the biggest problem with the argument of the 4% difference is that it is based on the fact that the RaTG13 strain is exactly what WIV declared, and that there are no other, closer strains in their collection. If we want to openly consider the laboratory leak hypothesis, we must admit that we cannot fully trust the data from the same laboratory that is suspected of leakage. If the leak did occur, as the hypothesis suggests, then the WIV is already trying to hide it, and the data they provide may well follow the same goal.</font></font><br>
<br>
<b>   ,      100%- ,    . ,   ,  , ,        ,    <i> </i>,      ,        .  ‚Äî     ‚Äî   CoV2    ,      ,         ,     ,     ,  .        , ,   ,        .    , ,   ,   ,   .</b><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, there is something that could help to believe WIV's claims about the nature of RaTG13 - if they agree to transfer their Yunnan samples from 2013, from which Shi Zhengli extracted RaTG13, to independent laboratories. </font><font style="vertical-align: inherit;">They still have to have them if they sequenced the complete RaTG13 genome from them again in 2020. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
****************************************.... ***************************</font></font><br>
<br>
<h3><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">UPD2: RaTG13 - the same strain as RaBtCoV / 4991?</font></font></h3><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
After I published this post, I was pointed out to a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">preprint</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> stating that RaTG13 may actually be a strain of RaBtCoV / 4991 ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">KP876546</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ), which Shi Zhengli </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">reported to</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> discover in 2013 in an abandoned Yunnan mine </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;">in 2016</font></a><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">There are several reasons to think so. </font><font style="vertical-align: inherit;">First of all, the only published sequence from RaBtCoV / 4991 is 100% identical to the </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">nucleotide</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> level RaTG13 sequence </font><font style="vertical-align: inherit;">, although this is only 370 nucleotides from the RdRp gene:</font></font><br>
<br>
<div style="text-align:center;"><img width="500" src="https://habrastorage.org/getpro/habr/post_images/171/996/21b/17199621b1f1a69be66efae600e785f5.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Secondly, the data on the collection of material from which these strains were isolated are almost identical: both were collected in July 2013 from a bat swab of </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">R. affinis</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> :</font></font><br>
<div style="text-align:center;"><img width="700" src="https://habrastorage.org/getpro/habr/post_images/017/9ca/e0e/0179cae0ee631abebbfb56967735e4ef.png"></div><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
A sample of RaBtCoV / 4991 was collected at a mine located in Mujiang County, which is under the jurisdiction of Puer:</font></font><br>
<blockquote><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Mujiang Autonomous Region is an autonomous region under the jurisdiction of the city of Puer, in the south of Yunnan, China. </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Wikipedia</font></font></a></i></blockquote><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">And the city of Puer is indicated as the gathering place of RaTG13 in the GISAID database, which may well be an approximate indication of the location of the mine in Mujiang. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
It is strange that in his </font><font style="vertical-align: inherit;">2020 </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on RaTG13, Shi Zhengli does not mention RaBtCoV / 4991 and does not cite his </font><font style="vertical-align: inherit;">2016 </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on its discovery, in which it is indicated as ‚Äúdeveloped and coordinated the study.‚Äù At the same time, RaBtCoV / 4991 is unlikely to be completely forgotten, since it is mentioned in an </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">article</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> by the Shi Zhengli group from 2019, where it is included in the phylogenetic tree of other coronaviruses:</font></font><br>
<br>
<div style="text-align:center;"><img width="700" src="https://habrastorage.org/getpro/habr/post_images/d75/526/ac8/d75526ac83572d09cf0195b20e2f33c4.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
I doubt that the place of RaBtCoV / 4991 in this tree was determined solely on the basis of a fragment of 370 nucleotides, so I think that by the beginning of 2019, the Shi Zhengli group had already sequenced its genome. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
By the way, it is interesting that the genomes of pangolin-2017 and pangolin-2019 are also very close to RaTG13 and CoV2 in this region of the RdRp gene, and CoV2 and pangolin-2019 have several common mutations not observed in RaTG13:</font></font><br>
<div style="text-align:center;"><img width="700" src="https://habrastorage.org/getpro/habr/post_images/b69/00a/4b4/b6900a4b4b532c948938971fa669c2d2.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
I also decided to compare the codon profile between RaTG13 and other strains from </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">R. affinis</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> living in the same abandoned mine. </font><font style="vertical-align: inherit;">Unfortunately, only 816-nucleotide segments from the RdRp gene ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">RaBtCoV / 3750</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> and </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">RaBtCoV / 4307‚Äì2</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ) </font><font style="vertical-align: inherit;">were published for other Ra strains </font><font style="vertical-align: inherit;">; therefore, to compare codon preferences, I extracted the corresponding </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u=" rel="nofollow"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">816-nucleotide segment</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> from RaTG13. </font><font style="vertical-align: inherit;">As a result, RaTG13 is again noticeably different, while the other two strains are quite close:</font></font><br>
<br>
<div style="text-align:center;"><img width="700" src="https://habrastorage.org/getpro/habr/post_images/676/afa/33a/676afa33a702d71e3bd39ec2fc8272ac.png"></div></div>
      
    </div>
<section class="more-articles-navigation-panel js-more-articles-navigation-panel">
<h4>More articles:</h4>
<nav class="list-of-articles-container js-list-of-articles-container"><ul class="list-of-pages js-list-of-pages">
<li><a href="../en497940/index.html">Developer Heart: Sega Saturn Devkits</a></li>
<li><a href="../en497944/index.html">Using NAT Traversal to Connect Users Passively</a></li>
<li><a href="../en497946/index.html">Was there life before an Audio CD? PCM software decoder</a></li>
<li><a href="../en497950/index.html">21 lessons learned in 21 years of programming</a></li>
<li><a href="../en497952/index.html">English abbreviations and IT titles that we mispronounce</a></li>
<li><a href="../en497958/index.html">They are among us: exploring vulnerabilities and malicious code in Zoom for Windows</a></li>
<li><a href="../en497960/index.html">Linux network installation</a></li>
<li><a href="../en497964/index.html">Telemedicine. Reverse Engineering Electronic Stethoscope</a></li>
<li><a href="../en497968/index.html">Web Developer Resources: APIs and UI Elements</a></li>
<li><a href="../en497980/index.html">How much water has leaked? We solve the problem by a moonwalk on Haskell</a></li>
</ul></nav>
</section><br />
<a href="../../allArticles.html"><strong>All Articles</strong></a>
<script src="../../js/main.js"></script>

<!-- Yandex.Metrika counter -->
<script type="text/javascript" >
  (function (d, w, c) {
      (w[c] = w[c] || []).push(function() {
          try {
              w.yaCounter63335242 = new Ya.Metrika({
                  id:63335242,
                  clickmap:true,
                  trackLinks:true,
                  accurateTrackBounce:true,
                  webvisor:true
              });
          } catch(e) { }
      });

      var n = d.getElementsByTagName("script")[0],
          s = d.createElement("script"),
          f = function () { n.parentNode.insertBefore(s, n); };
      s.type = "text/javascript";
      s.async = true;
      s.src = "https://mc.yandex.ru/metrika/watch.js";

      if (w.opera == "[object Opera]") {
          d.addEventListener("DOMContentLoaded", f, false);
      } else { f(); }
  })(document, window, "yandex_metrika_callbacks");
</script>
<noscript><div><img src="https://mc.yandex.ru/watch/63335242" style="position:absolute; left:-9999px;" alt="" /></div></noscript>

<!-- Google Analytics -->
  <script>
    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
    ga('create', 'UA-134228602-13', 'auto'); ga('send', 'pageview')
  </script>
  <script src="https://www.google-analytics.com/analytics.js" async defer></script>

</section>

<footer class="page-footer">
  <div class="page-footer-legal-info-container page-footer-element">
    <p>
      Geek Week | <span class="page-footer-legal-info-year js-page-footer-legal-info-year">2020</span>
    </p>
  </div>
  <div class="page-footer-counters-container page-footer-element">
    <a class="page-footer-counter-clustrmap" href='#'  title='Visit tracker'><img src='https://clustrmaps.com/map_v2.png?cl=698e5a&w=271&t=t&d=i62cJ2037o_BACd40gCrIso3niu0Sjx2sDFYJkeYdRk&co=3a3a3a&ct=ffffff'/></a>
  </div>
</footer>
  
</body>

</html>