<!doctype html>
<html class="no-js" lang="en">

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-134228602-13"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-134228602-13');
  </script>

  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>ü§πüèΩ üö§ üêß ChIP-seq data analysis: from histones to computer tasks üçõ üòß üßëüèø‚Äçü§ù‚Äçüßëüèæ</title>
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <meta name="description" content="Every year, the Institute of Bioinformatics in St. Petersburg and Moscow recruits biologists, mathematicians and programmers to immerse themselves in ...">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <link rel="stylesheet" href="../../css/main.css">

  <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">

  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script>window.jQuery || document.write('<script src="../../js/vendors/jquery-3.3.1.min.js"><\/script>')</script>

  <script>document.write('<script src="//pagea' + 'd2.googles' + 'yndication.com/pagea' + 'd/js/a' + 'dsby' + 'google.js"><\/script>')</script>
  <script>
        var superSpecialObject = {};
        superSpecialObject['google_a' + 'd_client'] = 'ca-p' + 'ub-6974184241884155';
        superSpecialObject['enable_page_level_a' + 'ds'] = true;
       (window['a' + 'dsbygoogle'] = window['a' + 'dsbygoogle'] || []).push(superSpecialObject);
  </script>
</head>

<body>
  <!--[if lte IE 9]>
    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
  <![endif]-->
  <header class="page-header js-page-header">
    <a class="page-header-logo-container" href="https://geek-week.github.io/index.html"></a>
    <div class="page-header-text">Get best of the week</div>
  </header>
  <section class="page js-page"><h1>ChIP-seq data analysis: from histones to computer tasks</h1><div class="post__body post__body_full">
      <div class="post__text post__text-html post__text_v1" id="post-content-body" data-io-article-url="https://habr.com/ru/company/JetBrains-education/blog/489304/"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Every year, the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Institute of Bioinformatics</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in St. Petersburg and Moscow recruits biologists, mathematicians and programmers to immerse themselves in the world of bioinformatics. </font><font style="vertical-align: inherit;">Biologists learn to program and train to implement ideas in code, and computer scientists study biology and apply algorithmic approaches to biological and medical problems. </font><font style="vertical-align: inherit;">The most important part of training is real science projects. </font><font style="vertical-align: inherit;">In this article, we will talk about the work and results of students of the Institute, done under the direction of Oleg Shpynov from </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">JetBrains Research</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in 2019. </font><font style="vertical-align: inherit;">The project is devoted to the study of changes in human chromatin using machine learning. </font></font><br>
<br>
<img src="https://habrastorage.org/webt/ax/ru/yp/axruypfh7nrjt52ngpsvxjzxqei.jpeg"><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Informatics students 2019 Institute of Bioinformatics</font></font></i> <br>
<a name="habracut"></a><br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">What is sequencing and why is it needed</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The desire to satisfy curiosity and understand oneself, which began with a description of human anatomy, gradually deepened and moved to a more detailed level. Blood cells and their interaction with parasites, the mechanisms of transmission of hereditary information and the formation of metastases by cancer cells were studied.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The advent of sequencing technologies has allowed us to go one more level deeper and look directly ‚Äúin the face‚Äù of the carrier of genetic information - DNA. </font><font style="vertical-align: inherit;">In other words, deoxyribonucleic acid, which is located in the nucleus of almost every cell in our body, is responsible for how we look, how tall, what timbre of voice we speak and whether we can get malaria. </font><font style="vertical-align: inherit;">However, technology, like biochemical methods, does not stand still. </font><font style="vertical-align: inherit;">Their combination made it possible to "bring to light" more complex mechanisms of the body. </font><font style="vertical-align: inherit;">Let's deal with this in more detail.</font></font><br>
<br>
<h4><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">How do we sequence organisms</font></font></h4><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Sequencing technologies have changed, and now technological progress allows, depending on the wishes, sequencing of separate cells, watching changes in them over time, or simply getting complete information about the sequence of the carrier of hereditary information - DNA. </font><font style="vertical-align: inherit;">In fact, sequencing allows you to translate a biological molecule into a text file, which you can then work with as plain text. </font><font style="vertical-align: inherit;">Modern sequencing methods use the ‚Äúshotgun‚Äù approach and yield a huge number of short fragments. </font><font style="vertical-align: inherit;">In some analyzes, these short fragments are ‚Äútried on‚Äù on existing genomes and look at the differences in the sequence of the ‚Äútext‚Äù.</font></font><br>
<br>
<h4><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">What are histones and what do they affect</font></font></h4><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The DNA strand is very long and can‚Äôt be permanently in an untwisted state - it is inconvenient and dangerous (there is a greater likelihood of a gap somewhere). Therefore, the molecule spirals (twists very strongly) and is compactly packed, wrapped in special protein complexes, like hair on curlers. These proteins are called nucleosomes and are composed of histone proteins. Histone modification is one example of a more general mechanism of epigenetic regulation. The organism is alive and needs to respond to surrounding changes. The reaction of the body is including the change in gene expression. If the DNA fragment on which the gene is located is tightly packed and wound on the nucleosome, then it is impossible to get to it and read the information. Therefore, special phosphoryl and acetyl groups are hung on histones,so-called phosphorylation or acetylation occurs. This causes the histone to "move" and give access to the desired DNA fragment. But the nucleosome still remains bound to DNA and this can be used in regulatory studies.</font></font><br>
<br>
<img src="https://habrastorage.org/webt/rm/jr/6e/rmjr6ezyz486qljiqsmujattkt0.png"><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">The mechanism of acetylation and methylation of histones</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">source</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> )</font></font><br>
<br>
<h4><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Chromatin-immunoprecipitation sequencing (ChIP-seq) and its use</font></font></h4><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
To study the DNA fragments that remain bound to the protein, there is a special method: chromatin immunoprecipitation (chromatin immunoprecipitation, ChIP). </font><font style="vertical-align: inherit;">This analysis takes place as follows:</font></font><br>
<br>
<ul>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">reversible crosslinking between DNA and its interacting proteins (usually by formaldehyde treatment) </font></font></li>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> DNA isolation and fragmentation by ultrasound or endonucleases</font></font></li>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> protein-specific antibody deposition</font></font></li>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">the destruction of cross-links between protein and DNA, DNA purification </font></font></li>
</ul><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In short, we remove the protein linked to the DNA from the solution and make it ‚Äúlet go‚Äù of the DNA. From a biological point of view, the field of action is understandable: the study of gene expression, closed and open areas, etc. We'll talk about the things that programmers can do in this task below. </font></font><br>
 <br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the case of ChIP sequencing (-seq), the resulting DNA fragments are amplified (artificial duplication of fragments) and sequenced. Set of sequences of small pieces of DNA and study bioinformatics. </font></font><br>
 <br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The received data passes quality control, is filtered, aligned to a DNA sequence and processed by special programs. </font></font><br>
<br>
<img src="https://habrastorage.org/webt/ar/ix/uj/arixujqmye6j1zbtyfb6o8nxnky.png"><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">DNA Preparation Scheme for Analysis</font></font></i><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The task of finding DNA binding sites is often called the peak calling task, and the tool class is peak callers. </font><font style="vertical-align: inherit;">At the moment, there are many computational approaches and tools for analyzing such data, however, the algorithms are not ideal and have a number of limitations. </font><font style="vertical-align: inherit;">There are still many unsolved computational problems for programmers and computer scientists in this area. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Here are some of them that students of mathematical and technical specialties are currently solving:</font></font><br>
<br>
<ul>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Uneven fragmentation and control</font></font><br>
 </li>
</ul><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The availability of chromatin during fragmentation is not the same in different parts of the genome: it is more accessible in actively transcribed regions, therefore, the corresponding DNA fragments will prevail in the sample, which can lead to a false-positive result. </font><font style="vertical-align: inherit;">Tightly packed areas, in contrast, may be less likely to fragment and therefore be less represented in the sample, which can lead to a false negative result.</font></font><br>
<br>
<ul>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> Number of cells</font></font></li>
</ul><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The classical technique has a number of limitations. </font><font style="vertical-align: inherit;">So, usually a significant number of cells (about 10 million) are needed for ChIP-seq, which complicates the application of this method on small organisms (such as fungi or protozoa), and also limits the number of experiments that can be performed with a valuable sample.</font></font><br>
<br>
<ul>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Data noise </font></font></li>
</ul><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
During the ChIP-seq experiment, it is possible to obtain in the final library not only DNA fragments that were associated with the protein, but also other, non-specifically related fragments. This may occur due to not ideal specificity of the antibody, problems with washing free DNA fragments, etc. Such fragments form the so-called noise in the data. The problem lies not only in the existence of noise, but also in the complexity of its measurement. To assess its level, there is a signal-to-noise ratio (SNR) metric, which is determined by the number and power of the peaks obtained for each sample. However, a high SNR does not guarantee the correct determination of binding sites, but merely reflects the presence of a large number of genome regions,which are aligned (on the chromosome in this place the sequence coincides with the desired) many reads - small fragments of DNA.</font></font><br>
<br>
<h4><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Problem Solving Options</font></font></h4><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Part of these tasks was solved by students of the Institute of Bioinformatics under the direction of Oleg Shpynov from JetBrains Research as part of semester research projects. </font></font><br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Noisy peak calling. </font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
student: Chaplygina Daria</font></font><br>
<br>
<img src="https://habrastorage.org/webt/cs/ww/ci/cswwciwpijbtocqmd7lmsdzjwu4.jpeg"><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the article ‚ÄúImpact of sequencing depth in ChIP-seq experiments‚Äù (1), the authors studied the effect of library size (the number of initial reads) on the results of peak search algorithms. They created artificial datasets for different types of histone modifications by random sampling from real experiments. As expected, the poorer the library, the more difficult it is for the algorithms to find peaks, the results are inconsistent between different methods. But they also noticed that, in the case of using the same tool, the coordination between biological replicates is lost. In a semester project, we investigated the effect of noise in the source data. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The data set with a controlled noise level was obtained on the basis of publicly available data from ChIP-seq experiments from the ENCODE project site</font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">ENCODE project</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">Two noise models were used for this:</font></font><br>
<br>
<ol>
<li><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Additive model. </font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Fragments from random sections of DNA were added to the source file with ‚Äúclean data‚Äù. </font><font style="vertical-align: inherit;">The proportion of random fragments ranged from 0% to 90%.</font></font></li>
<li><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Probabilistic Model. </font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">For each experiment, a mathematical model was built using the Tulip tool. </font><font style="vertical-align: inherit;">With its help, a completely new experiment was generated, one of the parameters of which - the percentage of fragments that are located inside the DNA-protein binding sites - varied from 10% to 0.5%.</font></font><br>
 </li>
</ol><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Probabilistic Model. For each experiment, a mathematical model was built using the Tulip tool. With its help, a completely new experiment was generated, one of the parameters of which - the percentage of fragments that are located inside the DNA-protein binding sites - varied from 10% to 0.5%. </font></font><br>
<br>
<img src="https://habrastorage.org/webt/xw/ri/24/xwri24184ad854ssuzd2kak96mg.png"><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Visualization of data changes when applying a probabilistic noise model</font></font></i><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
On the obtained data set, we analyzed three algorithms: </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">MACS2</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (2), </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">SICER</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (3) and </font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">SPAN</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (an algorithm developed by JetBrains Research. It is based on </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">semi-supervised</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">machine learning method). As it turned out, with a fixed SNR, one can predict the expected accuracy and completeness of the set of peaks that will be found by the algorithm. At a high noise level (or low SNR): MACS2 and SICER almost do not find peaks, while SPAN shows the most stable results in terms of a combination of indicators. </font></font><br>
<br>
<img src="https://habrastorage.org/webt/qc/_l/6h/qc_l6howtxruxibmbpbnnnb2wec.png"><br>
<img src="https://habrastorage.org/webt/3b/7x/sv/3b7xsvqw_lnexkipkupjdipivnq.png"><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Accuracy and completeness of peak search algorithms in a controlled noise level</font></font></i><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
We studied how in the process of noisiness two metrics of data quality change: SNR and percentage of fragments inside peaks (FRIP - Fraction of Reads In Peaks). The measurements showed that for the same SNR, the fraction of fragments per region of DNA ‚Äì protein interaction can vary significantly (in some cases, the difference was up to 50%). Existing standards and recommendations for assessing the quality of these ChIP-seq experiments are incomplete, and new integrated approaches are required. </font></font><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
As part of the work, we also developed pipelines for semi-automatic conducting such experiments. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Implementation of approaches and source code: </font></font><br>
<br>
<a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">github.com/DaryaChaplygina/NoisyPeakCalling</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , </font></font><br>
<br>
<a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">github.com/DaryaChaplygina/NoisyPeakCalling2</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> .</font></font><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Deep learning to the rescue!</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
student: Daria Balashova </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
One of the limitations of the classical ChIP-seq method is the large amount of necessary cellular material, which does not allow the experiment, for example, in the case of rare cell populations or in the case of several measurements for one biological sample. The new ChIP-seq (4) Ultra-Low-Input (ULI) method requires significantly less material ‚Äî 100,000 cells are sufficient ‚Äî but has greater variability and noise level in the data. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The use of deep machine learning methods is gaining popularity in bioinformatics, demonstrating excellent results in solving problems such as processing biomedical images. In the work ‚ÄúDenoising genome-wide histone ChIP-seq with convolutional neural networks‚Äù (5), the authors proposed an algorithm</font></font><b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Coda</font></font></b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> is a method of improving the quality of ChIP-seq data based on convolutional neural networks. They created and trained a deep neural network not only to improve poor quality data, but also to find peaks in them. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the framework of this project, the original algorithm was adapted for ULI ChIP-seq data. Using the findings from the previous project and the ULI ChIP-seq data from the article ‚ÄúEpigenetic changes in aging human monocytes‚Äù (6), we analyzed such important characteristics of the algorithm as improving quality metrics, for example, SNR. As a result, the </font><b><i><font style="vertical-align: inherit;">DCNN</font></i></b><font style="vertical-align: inherit;"> algorithm was created </font></font><b><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">.</font></font></i> </b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">- convolutional neural network to automatically improve the quality of the data based on the signal-to-noise ratio in the case of biological repetitions. </font><font style="vertical-align: inherit;">If improvement and signal purification works quite well, then the search for binding sites of proteins with DNA using deep learning methods is still an unresolved problem, since existing approaches require a large and high-quality training sample. </font></font><br>
<br>
<img src="https://habrastorage.org/webt/ka/qx/3c/kaqx3cgf8-w3tymzwq3me4y1kny.png"><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Schematic representation of the application of the convolutional neural network DCNN</font></font><br>
</i><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
Implementation of the approach and source code: </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">github.com/dashabalashova/Denoising_CNN</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> .</font></font><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Instead of an afterword</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Bioinformatics allows you to apply the approaches of programmers to biological data and gain new knowledge that will help biologists and doctors to study humans. </font><font style="vertical-align: inherit;">Now open is accepting applications for the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">summer school 2020</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , which will be held in St. Petersburg from July 27 to August 1. </font><font style="vertical-align: inherit;">It is ideal for exploring bioinformatics. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
For those who have decided on a more serious training - there is a chance to jump into the last car and apply for a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">retraining program in bioinformatics</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> in St. Petersburg and Moscow until </font><font style="vertical-align: inherit;">February 22, </font><font style="vertical-align: inherit;">or until March 1 at the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">retreat seminar on systems biology</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
For those who like to read and discover new things, we have a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">list of books and textbooks</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> on algorithms, programming, genetics and biology.</font></font><br>
<br>
<b><h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Bibliography:</font></font></h2></b><br>
<ol>
<li> Jung, Y. L., Luquette, L. J., Ho, J. W., Ferrari, F., Tolstorukov, M., Minoda, A.,‚Ä¶ &amp; Park, P. J. (2014). Impact of sequencing depth in ChIP-seq experiments. Nucleic acids research, 42(9), e74-e74.<br>
</li>
<li>Zhang, Y., Liu, T., Meyer, C. A., Eeckhoute, J., Johnson, D. S., Bernstein, B. E.,‚Ä¶ &amp; Liu, X. S. (2008). Model-based analysis of ChIP-Seq (MACS). Genome biology, 9(9), R137.<br>
 </li>
<li> Xu, S., Grullon, S., Ge, K., &amp; Peng, W. (2014). Spatial clustering for identification of ChIP-enriched regions (SICER) to map regions of histone methylation patterns in embryonic stem cells. In Stem Cell Transcriptional Networks (pp. 97-111). Humana Press, New York, NY.<br>
</li>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Brind'Amour, J., Liu, S., Hudson, M., Chen, C., Karimi, MM, &amp; Lorincz, MC (2015). </font><font style="vertical-align: inherit;">An ultra-low-input native ChIP-seq protocol for genome-wide profiling of rare cell populations. </font><font style="vertical-align: inherit;">Nature communications, 6 (1), 1-8.</font></font><br>
 </li>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Koh, PW, Pierson, E., &amp; Kundaje, A. (2017). </font><font style="vertical-align: inherit;">Denoising genome-wide histone ChIP-seq with convolutional neural networks. </font><font style="vertical-align: inherit;">Bioinformatics, 33 (14), i225-i233.</font></font><br>
</li>
<li><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Schukina, Bagaitkar, Shpynov et al., In review, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">artyomovlab.wustl.edu/aging</font></font></a> </li>
</ol><br>
<br>
<b><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Authors of the article:</font></font></b><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
Olga Bondareva, Institute of Bioinformatics </font></font><br>
<a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Oleg Shpinov</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , JetBrains Research </font></font><br>
<a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Ekaterina Vyakhhi</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , Institute of Bioinformatics</font></font></div>
      
    </div>
<section class="more-articles-navigation-panel js-more-articles-navigation-panel">
<h4>More articles:</h4>
<nav class="list-of-articles-container js-list-of-articles-container"><ul class="list-of-pages js-list-of-pages">
<li><a href="../en489290/index.html">"Work in IT - 2020", or Who and for what we love</a></li>
<li><a href="../en489294/index.html">Learning CSS Variables with a Practical Example</a></li>
<li><a href="../en489296/index.html">Functional options on steroids</a></li>
<li><a href="../en489298/index.html">An informal interview with Maria Dolgusheva (HR from Silicon Valley)</a></li>
<li><a href="../en489302/index.html">Issue # 31: IT training - current issues and challenges from leading companies</a></li>
<li><a href="../en489306/index.html">[Habr]: Donuts</a></li>
<li><a href="../en489310/index.html">Top 10: the best reports of Heisenbug 2019 Moscow</a></li>
<li><a href="../en489314/index.html">Ethernet Quartet: Old Speed, New Features</a></li>
<li><a href="../en489318/index.html">Mitap "DevOps Scalable Systems"</a></li>
<li><a href="../en489320/index.html">Practice using AI. Workshop ‚ÄúAnalysis of drone images to identify damaged houses‚Äù</a></li>
</ul></nav>
</section><br />
<a href="../../allArticles.html"><strong>All Articles</strong></a>
<script src="../../js/main.js"></script>

<!-- Yandex.Metrika counter -->
<script type="text/javascript" >
  (function (d, w, c) {
      (w[c] = w[c] || []).push(function() {
          try {
              w.yaCounter63335242 = new Ya.Metrika({
                  id:63335242,
                  clickmap:true,
                  trackLinks:true,
                  accurateTrackBounce:true,
                  webvisor:true
              });
          } catch(e) { }
      });

      var n = d.getElementsByTagName("script")[0],
          s = d.createElement("script"),
          f = function () { n.parentNode.insertBefore(s, n); };
      s.type = "text/javascript";
      s.async = true;
      s.src = "https://mc.yandex.ru/metrika/watch.js";

      if (w.opera == "[object Opera]") {
          d.addEventListener("DOMContentLoaded", f, false);
      } else { f(); }
  })(document, window, "yandex_metrika_callbacks");
</script>
<noscript><div><img src="https://mc.yandex.ru/watch/63335242" style="position:absolute; left:-9999px;" alt="" /></div></noscript>

<!-- Google Analytics -->
  <script>
    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
    ga('create', 'UA-134228602-13', 'auto'); ga('send', 'pageview')
  </script>
  <script src="https://www.google-analytics.com/analytics.js" async defer></script>

</section>

<footer class="page-footer">
  <div class="page-footer-legal-info-container page-footer-element">
    <p>
      Geek Week | <span class="page-footer-legal-info-year js-page-footer-legal-info-year">2020</span>
    </p>
  </div>
  <div class="page-footer-counters-container page-footer-element">
    <a class="page-footer-counter-clustrmap" href='#'  title='Visit tracker'><img src='https://clustrmaps.com/map_v2.png?cl=698e5a&w=271&t=t&d=i62cJ2037o_BACd40gCrIso3niu0Sjx2sDFYJkeYdRk&co=3a3a3a&ct=ffffff'/></a>
  </div>
</footer>
  
</body>

</html>