<!doctype html>
<html class="no-js" lang="en">

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-134228602-13"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-134228602-13');
  </script>

  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>üßòüèæ üå¨Ô∏è üñäÔ∏è What are the real math problems when developing vaccines from COVID-19? üíÜüèø üåé üöü</title>
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
  <meta name="description" content="The World Health Organization's updated list last week already lists 131 vaccine development projects designed to form an immune response to the SARS-...">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <link rel="stylesheet" href="../../css/main.css">

  <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">

  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script>window.jQuery || document.write('<script src="../../js/vendors/jquery-3.3.1.min.js"><\/script>')</script>

  <script>document.write('<script src="//pagea' + 'd2.googles' + 'yndication.com/pagea' + 'd/js/a' + 'dsby' + 'google.js"><\/script>')</script>
  <script>
        var superSpecialObject = {};
        superSpecialObject['google_a' + 'd_client'] = 'ca-p' + 'ub-6974184241884155';
        superSpecialObject['enable_page_level_a' + 'ds'] = true;
       (window['a' + 'dsbygoogle'] = window['a' + 'dsbygoogle'] || []).push(superSpecialObject);
  </script>
</head>

<body>
  <!--[if lte IE 9]>
    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
  <![endif]-->
  <header class="page-header js-page-header">
    <a class="page-header-logo-container" href="https://geek-week.github.io/index.html"></a>
    <div class="page-header-text">Get best of the week</div>
  </header>
  <section class="page js-page"><h1>What are the real math problems when developing vaccines from COVID-19?</h1><div class="post__body post__body_full">
      <div class="post__text post__text-html post__text_v1" id="post-content-body" data-io-article-url="https://habr.com/ru/company/hsespb/blog/505846/"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">The </font><font style="vertical-align: inherit;">World Health Organization's </font><font style="vertical-align: inherit;">updated </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">list</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> last week </font><font style="vertical-align: inherit;">already </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;">lists</font></a><font style="vertical-align: inherit;"> 131 vaccine development projects designed to form an immune response to the SARS-CoV-2 virus, the cause of COVID-19 disease. Among them there are two projects from our partner - the Russian biopharmaceutical company </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">BIOCAD</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . Getting on this list is not only a matter of prestige, but also the opportunity to join the working groups of vaccine developers around the world, discussing without question the questions of cellular and animal models, on which one or another vaccine could be tested. Nevertheless, information on how vaccines are developed is extremely sparingly beyond the boundaries of such specialized groups.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The media is actively promoting discussions of clinical research, production and future use of vaccines, while the "early" developers, rather, dismissed excessive publicity. Now the whole part on the design and assembly of the candidates is completed, we hid in anticipation of the results from animal models and obtaining permissions in public. So you can take a little time to tell what tasks arise inside. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Since Habr is primarily a near-resource, and in modern pharmaceuticals the methods of bioinformatics and computational biology are actively used, we will focus on ‚Äúdry‚Äù mathematical problems. In fact, these tasks were also chosen because next year the </font></font><strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Higher School of Economics - St. Petersburg and BIOCAD will open their Masters in Computational Biology</font></font></strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">but more about that at the very end. </font><font style="vertical-align: inherit;">First of all, we will figure out what, in fact, had to be done.</font></font><br>
<br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/496/9b2/0ba/4969b20ba974471a8b59dbafcc663b0e.png"></div><br>
<a name="habracut"></a><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">A bit about the authors</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Three BIOCAD employees worked on this article: </font></font><br>
<br>
<strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Diana Kondinskaya</font></font></strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , a graduate of biophysics at the Physics Department of St. Petersburg State University, and a senior specialist in structural bioinformatics. </font><font style="vertical-align: inherit;">With his own hands, he makes protein designs of original drugs, he is engaged in molecular simulations and everything connected with them. </font></font><br>
<br>
<strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Alexandra Chertkova</font></font></strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> is a graduate of bio-informatics at SPbSPU. </font><font style="vertical-align: inherit;">He holds the position of head of the department of systems biology, is responsible for modeling the pharmacokinetics and pharmacodynamics of all preclinical and clinical studies of the company. </font></font><br>
<br>
<strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Pavel Yakovlev</font></font></strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">graduated from the bioinformatics program at Academic University. </font><font style="vertical-align: inherit;">During the implementation of the first research project at the university, he ended up in BIOCAD, being the first bioinformatics there. </font><font style="vertical-align: inherit;">He successively passed the positions of head of the bioinformatics group and director of the department of computational biology. </font><font style="vertical-align: inherit;">Since February 2020, he began to lead all research in the field of the development of original drugs as a director for early development and development.</font></font><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">What are vaccines?</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In simple terms, vaccines are a type of drug whose task is to ‚Äúteach‚Äù the immune system to find and destroy certain pathogens. Traditionally, this refers to preventive vaccines that train the immune system even before the pathogen enters the body, but in some areas there are exceptions, for example, therapeutic </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">oncological vaccines</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Briefly speaking about the functioning of the immune system of higher mammals is a thankless task. The immunology course at any biofacus is traditionally considered one of the most difficult, and it usually becomes obsolete faster than a semester. So, in no case claiming completeness, we turn only to the main details of </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;">adaptive immunity that</font></a><font style="vertical-align: inherit;"> interest us</font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">produced by the body individually to new threats. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The first important element is </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">B-cell</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">immunity. Its essence lies in the constant (throughout life!) The formation of new cells called B-lymphocytes. Each such lymphocyte is able to recognize a unique target using its B-cell receptor. Also, these cells are able to produce a special form of these receptors - immunoglobulins (or antibodies), which do not remain on the surface of the cell, but float freely, specifically binding pathogenic proteins both in free form and on the surface of the cells. Such antibodies can not only block any interaction, but also mark the cell for destruction by other components of the immune system - phagocytes and natural killers. By the way, synthetic antibodies </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">are the basis</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> for modern anti-cancer therapy.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The presence of a B-cell immune response implies some ‚Äúfriend or foe‚Äù system built into the body, which would allow the identification of external pathogens and critical mutations of their own proteins and trigger an immune response to them. Indeed, absolutely every cell in our body contains a system that provides a presentation of the proteins produced by the cell on the surface. So, a special protein complex, called the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">proteasome</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , is able to break down proteins inside the cell into short linear fragments - peptides consisting of no more than a couple of tens of amino acids. Such peptides can bind to </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">HLA</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> allele products </font><font style="vertical-align: inherit;">encoding specific </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;">MHC</font></a><font style="vertical-align: inherit;"> proteins</font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">(in this case, type I). Despite the fact that several thousand of these alleles are in the human genome (see, for example, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">here</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ), only a few hundred are actively working, and each person has his own. This is determined by both heredity and external factors, so it is usually fair to assume that the residents of one region will have the majority of active MHCs the same. Contacting with the products of proteasome degradation, type I MHCs enter the cell membrane, thus showing the products produced by the cell. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In addition to the mechanism for presenting their insides with each cell, there are also professional demonstrators - </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">antigen-presenting cells</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">. They are able to capture objects from the external environment and show their peptides in a similar way, using for this already </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">class II MHC</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , which are also HLA products. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
But since someone is demonstrating products, some system must recognize them. The second important branch of the adaptive immune system is responsible for this - the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">T-cell</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">immunity. With the help of special receptors (called, obviously, T-cell), such cells are able to recognize peptides that do not correspond to the normal proteins of the body and cause various immune reactions. T helper cells will look at complexes from class II MHC and the peptides they present and activate B-cell immunity and a number of other reactions. Cytotoxic cells T-killers will simply destroy everyone whose pair of MHC class I and the presented peptide they do not like.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Armed with this knowledge, we can now realize the true goal of the vaccine - to ensure the presence of pathogen proteins (in our case, the SARS-CoV-2 virus) in the body to start the production of B- and / or T-cell immunity, which will be able to quickly neutralize the pathogen itself when it enters the body. The produced antibodies will prevent the virus from entering the cell, and T-lymphocytes will be able to kill cells in which the virus has already settled. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Very often, weakened pathogenic microorganisms are used as vaccines - viruses and bacteria that carry characteristic proteins recognized by the immune system, but are harmless in themselves. Such vaccines are called </font></font><strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">attenuated.</font></font></strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">. However, developing and proving their safety is a complex and time-consuming process. </font></font><strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Recombinant vaccines</font></font></strong><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> created using genetic engineering techniques </font><font style="vertical-align: inherit;">can be used as an effective safe alternative </font><font style="vertical-align: inherit;">. In recombinant vaccines, only parts of the pathogenic organism are used, for example, a protein characteristic of it, for the recognition of which we want to tune the immune system. The WHO list mentioned at the beginning of the article contains only 9 attenuated vaccines, while the rest are various recombinant variants.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In the case of SARS-CoV-2 coronavirus, the SPIKE protein can serve as such a characteristic protein. It contains the RBD domain, which binds to the human ACE2 protein, through which the genetic material of the virus enters a healthy cell. If SPIKE or just RBD antibodies cling to it, the virus will lose the ability to leave its genetic material where it was not requested. Moreover, both this domain and the whole protein are quite conservative - that is, with all the speed of the virus change, the protein mutates very slowly. This means that vaccines based on this protein will remain effective for a long time. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Of course, without a fly in the ointment, nobody canceled the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">ADE effect</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">. </font><font style="vertical-align: inherit;">It lies in the fact that viruses associated with antibodies can begin to penetrate the cells of the human immune system and multiply in them, as they will be captured as pathogens. </font><font style="vertical-align: inherit;">Most often this is due to the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">steric interaction of the</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> antibody and the virus, as well as low </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">affinity</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (binding strength). </font><font style="vertical-align: inherit;">But still, if you try hard, you can make the vaccine effective and safe.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
From the foregoing, it is clear that you should work with the SPIKE protein or part of it. </font><font style="vertical-align: inherit;">But how to find this SPIKE? </font><font style="vertical-align: inherit;">How to find and select the desired domain from it? </font><font style="vertical-align: inherit;">And how to synthesize it for a vaccine so as not to overwhelm everyone who dared to work on it? </font><font style="vertical-align: inherit;">This is where the methods of rational design and bioinformatics will help. </font><font style="vertical-align: inherit;">Below we give some simple examples of specific tasks that can be solved with their help.</font></font><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Task 1: get the genetic constructs</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
So, we know that we want to work with the SPIKE protein or its part of RBD. </font><font style="vertical-align: inherit;">Protein can only be </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">obtained</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> using living cells that </font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;">synthesize it on an RNA matrix</font></a><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">Working with RNA is difficult, if only because it is very unstable, but the cells can get it themselves from </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">the DNA matrix</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">It is the DNA of the desired protein - its gene, that we can somehow place in the cells in order to get - to produce the desired protein. </font><font style="vertical-align: inherit;">How to get a gene? </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The SPIKE protein gene can be pulled directly from the virus using the polymerase chain reaction ( </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">PCR</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">) </font><font style="vertical-align: inherit;">But working with a real full-fledged virus is dangerous, for this it is necessary to implement and apply special security measures for employees and obtain a license for each room where such work will be carried out. </font><font style="vertical-align: inherit;">You can solve the problem differently: collect the desired gene from scratch.</font></font><br>
<br>
<div class="spoiler" role="button" tabindex="0">
                        <b class="spoiler_title"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">If you have forgotten how PCR works, this video allows you to quickly freshen up your memories.</font></font></b>
                        <div class="spoiler_text"> <iframe width="560" height="315" src="https://www.youtube.com/embed/gubLAtn2o4s" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe> </div>
                    </div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The length of the SPIKE gene is several thousand nucleotides. For PCR, this is nonsense, but since we abandoned the use of the viral genome, we do not have a ‚Äúmatrix‚Äù by which it can be synthesized. So, you need to do the assembly by chemical synthesis, collecting the entire gene for one nucleotide. And here difficulties begin: chemical synthesis will not pull such a length of structure. The probability of attaching the wrong nucleotide increases with increasing length: obtaining by this method a sequence of length N without a single error can be approximately estimated as 0.985 ^ N, so that, synthesizing the entire construct at once, we will get many errors. And many mistakes - this is not at all the gene that we wanted. But there is a way out: you can collect only small fragments (60‚Äì80 nucleotides each), and then </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">‚Äústick together‚Äù</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">them into the full gene using the same PCR. But, of course, it cannot do without nuances.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The first problem that we may encounter is the formation of a hairpin on a short single-stranded DNA fragment due to the complementary sticking of one part of it to another. Such a dense packing of a fragment can be inefficiently untwisted at PCR flow temperatures and, as a result, such a fragment will simply drop out of the reaction in which the target gene will no longer be obtained. Another problem may be the energy benefit of binding non-adjacent fragments. Thus, a product can be obtained from which a whole fragment of the gene of interest to us disappears, since fragments that are mostly distant from each other will ‚Äústick together‚Äù. Finally, the last simple (for understanding) problem of this kind may be the low energy of the complementary interaction of neighboring fragments. As a result, at high temperatures, PCR will not combine.and the final product will consist of several unrelated pieces, and not one target gene.</font></font><br>
<br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/7d5/e75/0ca/7d5e750caa70fca50a2e60c04c8100dd.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
An example of a tightly packed DNA fragment. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
In case we need to clearly preserve the nucleotide sequence of DNA that interests us, we have only one controlled optimization parameter - the length of the fragments from which the assembly is carried out. At the same time, we have three targets. In practice, we often can afford to make changes in the sequence itself.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The fact is that our main task is to obtain a protein that is determined by the sequence of amino acids. Proteinogenic amino acids are 20, and nucleotides 4. In 1961, Francis Watson and James Crick postulated that each triplet of nucleotides (codon) encodes one of twenty amino acids or a special terminal signal for terminating a protein - a stop codon. In total, there are 3 ^ 4 = 64 ways to encode 20 amino acids + 1 stop symbol. It turns out that each amino acid can be encoded in approximately three ways, and about 3 ^ N different genes can be used to obtain a protein of length N!</font></font><br>
<br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/9d7/ddc/bfc/9d7ddcbfc6a917392c5f299c5a116d61.png"></div><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Transition table from codons to amino acids.</font></font></i><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
So, changing codons to synonymous ones is another controllable optimization parameter. But here also appears a number of targeted ones. Not all genes will be equally willingly used by cells for protein synthesis. The efficiency of the translation process will depend on the packing density of the structure of the mRNA transcribed from the gene, as well as on the frequency of transport RNA present in this target organism that carries certain amino acids depending on the requested codon. So, having received a not-so-optimal gene, you can put an end to the production of the desired protein - whether it be SPIKE or RBD - in large quantities.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Here, several bioinformatic tasks arise at once. </font><font style="vertical-align: inherit;">First, it would be nice to be able to predict the structures of DNA and RNA, not only of individual molecules, but also, for example, of paired complexes. </font><font style="vertical-align: inherit;">Researchers have been working on this task for </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">almost 50 years</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , and fresh versions of algorithms </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">continue to be released</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> to this day. </font><font style="vertical-align: inherit;">Secondly, we have the problem of discrete combinatorial multi-parameter optimization in a huge space. </font><font style="vertical-align: inherit;">We </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">at one time developed</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> and continue to develop a tool that solves this problem.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Thus, it is possible to assemble any genetic construct that interests us, even a very large length. </font><font style="vertical-align: inherit;">But is it always necessary? </font><font style="vertical-align: inherit;">We have already discussed above that only the RBD domain of the SPIKE protein can be the subject of our interest. </font><font style="vertical-align: inherit;">Let's talk about it in more detail.</font></font><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Task 2: find RBD</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
We know that we want to teach the body to cling to the virus with antibodies, preventing it from entering the cells. And we even know that the virus SPIKE is responsible for the penetration of the virus. But this protein is a huge colossus sticking out of a </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">viral capsid</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">, and we have already mentioned that the most important part of it is the RBD domain that binds to the ACE2 receptor. Moreover, it is unlikely to be of much use if the human immune system recognizes the SPIKE protein, but does not interfere with it doing its dirty work (and yes, we still remember the ADE effect). So, let's find RBD and focus the immune system only on it, without disturbing it with extra SPIKE fragments. But this is not so simple: we need the three-dimensional structure of an individual RBD to be exactly the same as its structure as part of SPIKE. Otherwise, the developed immunity to RBD simply does not recognize it in the new environment.</font></font><br>
<br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/496/9b2/0ba/4969b20ba974471a8b59dbafcc663b0e.png"></div><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">SARS-CoV-2 virus. The trimers of SPIKE proteins on its surface are marked in red.</font></font></i><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
It is clear that in order to accomplish this task, it is necessary to study the structure of the SPIKE protein. Now, so much time after the epidemic began, it has already been resolved, and only a very lazy structural bioinformatics did not look at it. But when we were just starting work on developing a vaccine, no one had time to get it. It would seem that the situation is hopeless. But it was not there!</font></font><br>
<br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/427/cc2/0b9/427cc20b90d3e4f3fa49145bb887eacc.png"></div><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Structure of an isolated SPIKE trimer from SARS-CoV-2.</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> The lower part grows from the envelope of the virus, and the upper sticks out and interacts with the cells of the human body. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
SARS-CoV-2 virus has relatives, for example, SARS-CoV virus. It was actively studied in the 2000s due to the fact that it was this virus that was responsible for the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">outbreak of SARS in</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 2002-2004. It also belongs to the coronavirus family, and the three-dimensional structure of its SPIKE protein is known. A great gift was the fact that these relatives are very similar in sequence. And their RBD domain is generally almost the same. And this means that we could count on the fact that their structure would be very similar. So based on the structure of the old SPIKE, we built the structure of the new SPIKE we need by the method</font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">homologous folding</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (more about it at the courses on structural bioinformatics). Later it turned out that they really are very similar, so we did not lose in due time.</font></font><br>
<br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/5e6/f91/d9b/5e6f91d9b6a30c3153a1f0e672783bc6.png"></div><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">One isolated SPIKE protein with a highlighted red RBD domain.</font></font></i><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
So, we have a structure. Now we need to isolate RBD from it. Yes, not just isolating, but ensuring that it retains its structure separately from SPIKE. And a good tactic is to choose several options and then </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in silico</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (that is, programmatically, without conducting a ‚Äúwet‚Äù experiment in the laboratory) to check their stability. Options should not be selected by enumeration - the presence of the structure allows us to calculate which covalent, hydrophobic, and electrostatic interactions stabilize the structure of the domain. Focusing on their preservation, you can select potentially very stable candidates. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Before transferring to the laboratory, they can be checked for stability by searching for minimum energy and modeling using</font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">molecular dynamics</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">To do this, they need to be parameterized correctly ‚Äî to select mathematical equations and their parameters that will correctly and physically describe the behavior of these domains in time. </font><font style="vertical-align: inherit;">In general, these are quite lengthy calculations, but if you suddenly have a good computing cluster with a large number of GPUs, then this is very simple and fast. </font><font style="vertical-align: inherit;">We have one. </font><font style="vertical-align: inherit;">So after all the tricks, we know which RBD sequence will go into our vaccines.</font></font><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Task 3: immunogenic fragments</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Alas, not all vaccines can cram such a large domain. Although RBD is not SPIKE, it‚Äôs still a bit big - more than 250 amino acids, while in some vectors (part of the vaccine that is the ‚Äúcarrier‚Äù of the presented protein) it is permissible to use proteins with a maximum of 150 amino acids, and among the variety of such sequences in the virus you need to choose the most immunogenic .&nbsp;</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
We have already mentioned T-cell immunity and the fact that MHC II proteins on antigen-presenting cells can show small peptides to the immune system, stimulating its work. And in order to stimulate the immune system as efficiently as possible, you need to know which peptide paired with which MHC II will be the most immunogenic. In this case, we can use a very simple idea in constructing a vaccine: instead of a whole protein, we will show the body several immunogenic peptides (for sure) connected by degradable linkers. Antigen-presenting cells using protease proteins will cut such constructs into individual peptides and present them to T-lymphocytes. It looks cool, but, as usual, not everything is so simple.</font></font><br>
<br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/b1e/0d5/370/b1e0d53701511e63e1e967dd3c1e30b8.png"></div><br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">MHC class II in combination with the peptide VGSDWRFLRGYHQYA</font></font></i><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> 
The problem is that there can be many short peptides from a large protein ( </font></font><code>N ‚Äî K + 1</code><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">where N is the length of the protein and K is the length of the peptide, which can vary in the range of 9-15 amino acids). And the MHC class II variants, as we wrote above, are generally different from population to population, from race to race. And one of the tasks that bioinformatics have to solve is just the search for such peptides, the presentation of which on the cell surface using MHC II will cause a good immune response in this population. Of course, the best method would be to select a number of characteristic MHC molecules and analyze all possible peptides for binding to these molecules by </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">predicting intermolecular interactions</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . Of course, this is cheaper than conducting similar studies.</font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">in vitro</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (in a wet laboratory), but still a little expensive. In this regard, machine learning methods are being actively developed in this area that, according to the sequence of the peptide, predict which MHC it can contact. The accuracy of these methods still leaves much to be desired and does not allow obtaining results for all cases, but in conjunction with a priori biological knowledge, for each specific case the problem is usually still solved.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
And now we have a set of immunogenic peptides, and the following task appears: to connect these peptides together so that the proteases that cut proteins into peptides cut our polypeptide chain exactly where it is needed. </font><font style="vertical-align: inherit;">Indeed, if the incision will occur in a random place, class II MHCs will show immunity not what we wanted, but ‚Äúchimeric‚Äù peptides that are not related to SARS-CoV-2. </font><font style="vertical-align: inherit;">Since there are many different proteases (there is a feeling that we still don‚Äôt know), and there is even more data on them and their favorite places in the section, we use analytics and machine learning to solve this problem.</font></font><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">What next</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The development of a drug is not only the selection of a sequence that will act in any way. This is an understanding of the essence of such an action in living multicellular organisms, especially in humans. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
To do this, you need to know how much the drug is stored in the body and how its concentration is associated with the therapeutic effect. Of course, both can be measured. But taking measurements ‚Äî for example, taking a blood test or, worse, a biopsy ‚Äî is not always an easy task. So, a mouse most often dies after taking a blood sample, and we cannot destroy thousands of mice per study. Patients participating in a clinical trial do not want to be bled every fifteen minutes and can be understood.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
What does this mean? This means that there are few points. In this case, the initial doses are far from being fully studied, and mathematical models are required that show pharmacokinetics ‚Äî the concentration of the drug versus time, dose, and body characteristics ‚Äî and pharmacodynamics ‚Äî the effect of the current concentration of the drug on the target. These models are often a system of differential equations, the solutions of which are able to make good approximations and have predictive power. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Consider an example. Suppose we have a drug in the form of a tablet, the patient takes it, and after a while the active substance of the tablet enters the bloodstream. When we take blood from a person, we can determine the concentration of the active substance and get points in time. It looks something like this:</font></font><br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/827/a48/d78/827a48d7889337d3f70da1eca60a4397.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
&nbsp;Such a point function is very limited, and we have no idea what will happen at any time and, more importantly, at different dose levels (after all, we could choose something not optimal for the study), but we would like to have this idea .</font></font><br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/21d/a9e/829/21da9e829a980e14ccad7a4b23e57843.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Then we build the dependence on the basis of a simple assumption: the drug - a fixed dose - enters the body, is absorbed at a certain speed </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">k </font></font></i><i><sub><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">a</font></font></sub></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , then spreads through the bloodstream of volume </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">V</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> (which we see in the time-concentration graph) and at a certain rate </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Cl</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> from the body is displayed. </font><font style="vertical-align: inherit;">A change in concentration over time is a derivative of concentration over time. </font><font style="vertical-align: inherit;">Season this fact with a pinch of the </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">law of masses</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">, and differential equations are already looming. The solution of a system that obeys these laws and is close to the obtained points will reflect the real dependence of the concentration on time, that is, the pharmacokinetics (FC) of the drug. At the same time, changing the initial condition - the dose of the drug administered, we can show what kind of PK will be in this case - after all, the parameters </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">k </font></font></i><i><sub><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">a</font></font></sub></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> , </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">V</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> and Cl reflect processes and objects that are </font></font><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">supposedly</font></font></i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> dose-independent. Of course, in real-life problems such models can look much more complicated, taking into account the circulation of the drug through the tissues, the interaction of different drugs with each other, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">population features</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">the body of certain people (for example, with renal failure that interferes with the withdrawal of the drug from the body) and so on.&nbsp; </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
The same can be done with the effect. Again, consider an example when our drug affects the synthesis of a protein. Similarly, we draw a diagram, write down the equations taking into account the influence of concentration, and bring the solution closer to the data.</font></font><br>
<div style="text-align:center;"><img src="https://habrastorage.org/getpro/habr/post_images/499/9ba/117/4999ba117b9f5aa1bcb0804923054c28.png"></div><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Thus, we have the dependence of concentration on time and dose (if we change the initial condition) C (t) and the dependence of the effect on concentration and time E (t). </font><font style="vertical-align: inherit;">Combining these entities, we get the FKFD-model, which, for example, can show what dose will give this or that effect. </font><font style="vertical-align: inherit;">And all this - on your computer! </font><font style="vertical-align: inherit;">Once again, this is just the tip of the iceberg: complex biological systems are usually described by a whole chain of interconnected processes, and the drugs used often affect several components of such systems at once. </font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
An attentive reader might have noticed that all of the above was not about FC / PD modeling for vaccines. </font><font style="vertical-align: inherit;">Why? </font><font style="vertical-align: inherit;">In the case of the vaccine, it is not very clear what to consider as FC and how to measure PD. </font><font style="vertical-align: inherit;">However, </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">researchers do not give up</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">, and mathematical modeling claims to be in this area.</font></font><br>
<br>
<h2><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Epilogue</font></font></h2><br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Using easy-to-understand examples, we talked about some of the tasks that bioinformatics perform very quickly if a virus that spans the whole world is on the verge. However, in reality, we solve these and many other problems every day. No pandemic does not negate the fact that people suffer from cancer and autoimmune diseases, and children are born with severe genetic defects. The creation of drugs that can help such patients is no less important, and the tasks that arise in this case are usually even more complicated and require much more time for research.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
To create a therapy for such diseases, we develop both biological and low molecular weight compounds. The former require genetic constructs, optimization of the living cells and the determination of the complex physicochemical characteristics of the resulting proteins and other biological molecules (for example, we make therapeutic viruses for the treatment of </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">orphan diseases</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">) For the second, problems arise in the course of organic synthesis, permeability of cell membranes, stability in various fractions of microsomes, and so on. To create such large and small molecules, we apply rational design methods based on three-dimensional full-atomic models of molecules and their targets, calculate methods for linking therapeutic molecules to their targets, optimize chemical synthesis schemes, and predict the behavior of the drug in the body depending on the height and weight of the patient and much more.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
Of course, to create medicines, highly qualified specialists are needed, both in biological and chemical "wet" laboratories, and in the "dry" areas of computational biology and bioinformatics. In Russia, there are no universities that produce ready-made computational biologists focused on solving the problems of modern drug development. For a long time, we trained specialists within BIOCAD and raised more than one generation of sought-after specialists. Now the time has come to open a full-fledged program on teaching bioinformatics, and it will be taught to undergraduates of the St. Petersburg HSE. The start of the program is September 2021, but already in 2020 we will launch a pilot track within the framework of the existing program ‚Äú </font></font><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Programming and data analysis</font></font></a><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> ‚Äù.</font></font><br>
<br><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
We are looking forward to students with a strong physical and mathematical background, without chemical and biological training. </font><font style="vertical-align: inherit;">Teachers of the St. Petersburg HSE will provide the best courses in the country on algorithms, programming, data analysis, and industry employees will talk about the special chapters in physics, molecular biology, chemistry, as well as special courses on molecular simulations, structural bioinformatics algorithms, systems pharmacology, and other topics important to the field. </font><font style="vertical-align: inherit;">And, which is also important, we will talk about real industrial biological problems and teach how to use the acquired knowledge and skills to solve them. </font></font><br>
<br>
<i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">Soon, information about the new program will appear on the website of the </font></font></i><a href="https://translate.googleusercontent.com/translate_c?depth=1&amp;pto=aue&amp;rurl=translate.google.com&amp;sl=ru&amp;sp=nmt4&amp;tl=en&amp;u="><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">St. Petersburg School of Physics, Mathematics and Computer Science</font></font></i></a><i><font style="vertical-align: inherit;"><font style="vertical-align: inherit;"> . </font><font style="vertical-align: inherit;">Follow the news!</font></font></i></div>
      
    </div>
<section class="more-articles-navigation-panel js-more-articles-navigation-panel">
<h4>More articles:</h4>
<nav class="list-of-articles-container js-list-of-articles-container"><ul class="list-of-pages js-list-of-pages">
<li><a href="../en505814/index.html">Radarr, Jackett and the bot in the cart. Download torrents in a new way</a></li>
<li><a href="../en505824/index.html">Guide: Your Own L2TP VPN</a></li>
<li><a href="../en505826/index.html">How to get 100% vision and even more</a></li>
<li><a href="../en505834/index.html">Median Samples. Confidence Intervals and Comparison</a></li>
<li><a href="../en505838/index.html">FPGA technology for thousands of applications</a></li>
<li><a href="../en505850/index.html">C ++ Concept-Based Polymorphism in Product Code: PassManager in LLVM</a></li>
<li><a href="../en505856/index.html">From Brute-Force to the attempt on privacy - what SaaS providers face</a></li>
<li><a href="../en505860/index.html">Spring Boot, Hibernate and Kotlin for beginners step by step</a></li>
<li><a href="../en505870/index.html">How to promote mobile games and applications in Japan, Korea and China</a></li>
<li><a href="../en505872/index.html">History of unmanned vehicles</a></li>
</ul></nav>
</section><br />
<a href="../../allArticles.html"><strong>All Articles</strong></a>
<script src="../../js/main.js"></script>

<!-- Yandex.Metrika counter -->
<script type="text/javascript" >
  (function (d, w, c) {
      (w[c] = w[c] || []).push(function() {
          try {
              w.yaCounter63335242 = new Ya.Metrika({
                  id:63335242,
                  clickmap:true,
                  trackLinks:true,
                  accurateTrackBounce:true,
                  webvisor:true
              });
          } catch(e) { }
      });

      var n = d.getElementsByTagName("script")[0],
          s = d.createElement("script"),
          f = function () { n.parentNode.insertBefore(s, n); };
      s.type = "text/javascript";
      s.async = true;
      s.src = "https://mc.yandex.ru/metrika/watch.js";

      if (w.opera == "[object Opera]") {
          d.addEventListener("DOMContentLoaded", f, false);
      } else { f(); }
  })(document, window, "yandex_metrika_callbacks");
</script>
<noscript><div><img src="https://mc.yandex.ru/watch/63335242" style="position:absolute; left:-9999px;" alt="" /></div></noscript>

<!-- Google Analytics -->
  <script>
    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
    ga('create', 'UA-134228602-13', 'auto'); ga('send', 'pageview')
  </script>
  <script src="https://www.google-analytics.com/analytics.js" async defer></script>

</section>

<footer class="page-footer">
  <div class="page-footer-legal-info-container page-footer-element">
    <p>
      Geek Week | <span class="page-footer-legal-info-year js-page-footer-legal-info-year">2020</span>
    </p>
  </div>
  <div class="page-footer-counters-container page-footer-element">
    <a class="page-footer-counter-clustrmap" href='#'  title='Visit tracker'><img src='https://clustrmaps.com/map_v2.png?cl=698e5a&w=271&t=t&d=i62cJ2037o_BACd40gCrIso3niu0Sjx2sDFYJkeYdRk&co=3a3a3a&ct=ffffff'/></a>
  </div>
</footer>
  
</body>

</html>